{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "11e10461",
   "metadata": {
    "papermill": {
     "duration": 0.01191,
     "end_time": "2024-02-20T19:20:15.320006",
     "exception": false,
     "start_time": "2024-02-20T19:20:15.308096",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "# Prepare data for Pipeline"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "80621712",
   "metadata": {
    "papermill": {
     "duration": 0.00896,
     "end_time": "2024-02-20T19:20:15.339307",
     "exception": false,
     "start_time": "2024-02-20T19:20:15.330347",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "## Setup and settings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "40775dab",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-10-14T14:33:37.863523Z",
     "start_time": "2022-10-14T14:33:14.145712Z"
    },
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:15.351103Z",
     "iopub.status.busy": "2024-02-20T19:20:15.350802Z",
     "iopub.status.idle": "2024-02-20T19:20:19.951206Z",
     "shell.execute_reply": "2024-02-20T19:20:19.950705Z"
    },
    "papermill": {
     "duration": 4.607187,
     "end_time": "2024-02-20T19:20:19.952407",
     "exception": false,
     "start_time": "2024-02-20T19:20:15.345220",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Importing packages\n",
    "import os\n",
    "import yaml\n",
    "import logging\n",
    "import pandas as pd\n",
    "from Bio.Seq import Seq\n",
    "from qiime2 import Artifact, Visualization, Metadata\n",
    "from qiime2.plugins import demux\n",
    "from qiime2.plugins.cutadapt.methods import trim_paired, trim_single\n",
    "from qiime2.plugins.demux.methods import filter_samples"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5ae2b8f0",
   "metadata": {
    "papermill": {
     "duration": 0.038188,
     "end_time": "2024-02-20T19:20:19.997191",
     "exception": false,
     "start_time": "2024-02-20T19:20:19.959003",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "### Receiving the parameters\n",
    "\n",
    "The following cell can receive parameters using the [papermill](https://papermill.readthedocs.io/en/latest/) tool."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "d0a933ba",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-06-28T18:50:41.594341Z",
     "start_time": "2022-06-28T18:50:41.591544Z"
    },
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.020008Z",
     "iopub.status.busy": "2024-02-20T19:20:20.019657Z",
     "iopub.status.idle": "2024-02-20T19:20:20.023992Z",
     "shell.execute_reply": "2024-02-20T19:20:20.023282Z"
    },
    "papermill": {
     "duration": 0.009005,
     "end_time": "2024-02-20T19:20:20.025535",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.016530",
     "status": "completed"
    },
    "tags": [
     "parameters"
    ]
   },
   "outputs": [],
   "source": [
    "params_path = os.path.join('..', 'params', 'parameter-set-01.yaml')\n",
    "experiment_name = 'exp-01'\n",
    "base_dir = os.path.join('/', 'home', 'username', 'pipeline-dir')\n",
    "manifest_file = os.path.join(base_dir, 'data', 'raw', 'manifest', 'manifest.csv')\n",
    "replace_files = False\n",
    "trim = None\n",
    "metadata_file = None\n",
    "threads = 1\n",
    "demux_file = None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "8260a71b",
   "metadata": {
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.031105Z",
     "iopub.status.busy": "2024-02-20T19:20:20.030982Z",
     "iopub.status.idle": "2024-02-20T19:20:20.033782Z",
     "shell.execute_reply": "2024-02-20T19:20:20.033346Z"
    },
    "papermill": {
     "duration": 0.006392,
     "end_time": "2024-02-20T19:20:20.034697",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.028305",
     "status": "completed"
    },
    "tags": [
     "injected-parameters"
    ]
   },
   "outputs": [],
   "source": [
    "# Parameters\n",
    "experiment_name = \"jennefer-vs-paired-end-trim\"\n",
    "base_dir = \"/mnt/nupeb/rede-micro/redemicro-jennefer\"\n",
    "manifest_file = \"/mnt/nupeb/rede-micro/redemicro-jennefer/data/manifest.csv\"\n",
    "metadata_file = \"/mnt/nupeb/rede-micro/redemicro-jennefer/data/metadata.tsv\"\n",
    "class_col = \"class\"\n",
    "classifier_file = \"/mnt/nupeb/rede-micro/datasets/16S_classifiers_qiime2/silva-138-99-nb-classifier.qza\"\n",
    "replace_files = False\n",
    "phred = 20\n",
    "trunc_f = 0\n",
    "trunc_r = 0\n",
    "overlap = 12\n",
    "threads = 6\n",
    "trim = {\n",
    "    \"overlap\": 8,\n",
    "    \"forward_primer\": \"CCTACGGGRSGCAGCAG\",\n",
    "    \"reverse_primer\": \"GGACTACHVGGGTWTCTAAT\",\n",
    "}\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2ebdf942",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-08-18T15:40:31.070885Z",
     "start_time": "2023-08-18T15:40:30.611671Z"
    },
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.038595Z",
     "iopub.status.busy": "2024-02-20T19:20:20.038468Z",
     "iopub.status.idle": "2024-02-20T19:20:20.041350Z",
     "shell.execute_reply": "2024-02-20T19:20:20.040918Z"
    },
    "papermill": {
     "duration": 0.005941,
     "end_time": "2024-02-20T19:20:20.042313",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.036372",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "out_dir = os.path.join(base_dir, 'experiments', experiment_name, 'qiime-artifacts')\n",
    "img_folder = os.path.abspath(os.path.join(base_dir, 'experiments', experiment_name, 'imgs'))\n",
    "data_dir =  os.path.abspath(os.path.join(base_dir, 'data'))\n",
    "raw_data_dir =  os.path.abspath(os.path.join(data_dir, 'raw'))\n",
    "interim_data_dir =  os.path.abspath(os.path.join(data_dir, 'interim'))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a100233a",
   "metadata": {
    "papermill": {
     "duration": 0.001587,
     "end_time": "2024-02-20T19:20:20.045478",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.043891",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "### Defining names and paths"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "8dd097d2",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-06-28T18:50:41.632125Z",
     "start_time": "2022-06-28T18:50:41.619710Z"
    },
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.049302Z",
     "iopub.status.busy": "2024-02-20T19:20:20.049182Z",
     "iopub.status.idle": "2024-02-20T19:20:20.051810Z",
     "shell.execute_reply": "2024-02-20T19:20:20.051391Z"
    },
    "papermill": {
     "duration": 0.00553,
     "end_time": "2024-02-20T19:20:20.052652",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.047122",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "# Create path if it not exists\n",
    "if not os.path.isdir(out_dir):\n",
    "    os.makedirs(out_dir)\n",
    "    print(f'New qiime-artifacts folder path created: {out_dir}')\n",
    "\n",
    "# Create path if it not exists\n",
    "if not os.path.isdir(img_folder):\n",
    "    os.makedirs(img_folder)\n",
    "    print(f'New img folder path created: {img_folder}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "0edf1552",
   "metadata": {
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.056454Z",
     "iopub.status.busy": "2024-02-20T19:20:20.056332Z",
     "iopub.status.idle": "2024-02-20T19:20:20.061571Z",
     "shell.execute_reply": "2024-02-20T19:20:20.061273Z"
    },
    "papermill": {
     "duration": 0.00816,
     "end_time": "2024-02-20T19:20:20.062446",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.054286",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "manifest_df = pd.read_csv(manifest_file)\n",
    "n_directions = len(manifest_df['direction'].unique())\n",
    "if n_directions == 1:\n",
    "    d_type = 'SampleData[SequencesWithQuality]'\n",
    "    v_type = 'SingleEndFastqManifestPhred33'\n",
    "    direction = 'single'\n",
    "elif n_directions == 2:\n",
    "    d_type = 'SampleData[PairedEndSequencesWithQuality]'\n",
    "    v_type = 'PairedEndFastqManifestPhred33'\n",
    "    direction = 'paired'\n",
    "else:\n",
    "    print(f'ERROR: invalid number of directions {n_directions}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "826a3f25",
   "metadata": {
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.066413Z",
     "iopub.status.busy": "2024-02-20T19:20:20.066293Z",
     "iopub.status.idle": "2024-02-20T19:20:20.069135Z",
     "shell.execute_reply": "2024-02-20T19:20:20.068724Z"
    },
    "papermill": {
     "duration": 0.005897,
     "end_time": "2024-02-20T19:20:20.070068",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.064171",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "sufix = str(direction)\n",
    "if trim and len(trim.keys()) == 3:\n",
    "    sufix += f'-{trim}'\n",
    "    \n",
    "# Define the output artifact full path\n",
    "if demux_file is None:\n",
    "    demux_file = os.path.join(out_dir, f'demux-{direction}.qza')\n",
    "demux_view = os.path.join(out_dir, f'demux-{direction}.qzv')\n",
    "demux_file_trim = os.path.join(out_dir, f'demux-{direction}-trim.qza')\n",
    "demux_view_trim = os.path.join(out_dir, f'demux-{direction}-trim.qzv')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dd85e68c",
   "metadata": {
    "papermill": {
     "duration": 0.001679,
     "end_time": "2024-02-20T19:20:20.073401",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.071722",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "## Step execution\n",
    "\n",
    "This Step import all `fastq` files in a **QIIME2 Artifact** object and save it to a new `qza` file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "f1099a3c",
   "metadata": {
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.077419Z",
     "iopub.status.busy": "2024-02-20T19:20:20.077235Z",
     "iopub.status.idle": "2024-02-20T19:20:20.082546Z",
     "shell.execute_reply": "2024-02-20T19:20:20.082114Z"
    },
    "papermill": {
     "duration": 0.008312,
     "end_time": "2024-02-20T19:20:20.083459",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.075147",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "# If the metadata file is not defined, use the default metadata file\n",
    "metadata_qa = Metadata.load(metadata_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "3c3e2e93",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-06-28T19:44:58.640338Z",
     "start_time": "2022-06-28T18:50:41.633657Z"
    },
    "execution": {
     "iopub.execute_input": "2024-02-20T19:20:20.087603Z",
     "iopub.status.busy": "2024-02-20T19:20:20.087285Z",
     "iopub.status.idle": "2024-02-20T19:59:57.240466Z",
     "shell.execute_reply": "2024-02-20T19:59:57.239991Z"
    },
    "papermill": {
     "duration": 2377.156527,
     "end_time": "2024-02-20T19:59:57.241636",
     "exception": false,
     "start_time": "2024-02-20T19:20:20.085109",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/lauro/anaconda3/envs/qiime2-2023.5/lib/python3.8/site-packages/q2_demux/_summarize/_visualizer.py:191: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  context['result_data'] = context['result_data'].append(df)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/lauro/anaconda3/envs/qiime2-2023.5/lib/python3.8/site-packages/q2_demux/_summarize/_visualizer.py:191: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  context['result_data'] = context['result_data'].append(df)\n"
     ]
    }
   ],
   "source": [
    "# If the demux file does not exist or if the old file will be replaced\n",
    "if not os.path.isfile(demux_file) or replace_files:\n",
    "    \n",
    "    interim_demux_path = os.path.join(interim_data_dir, f'demux-{direction}.qza')\n",
    "    \n",
    "    # Create new file path for interim dir\n",
    "    if not os.path.isdir(interim_data_dir):\n",
    "        !mkdir -p {interim_data_dir}\n",
    "        \n",
    "    # Load Artifact from interim folder\n",
    "    if os.path.isfile(interim_demux_path):\n",
    "        artifact = Artifact.load(interim_demux_path)\n",
    "        \n",
    "    # Create new Artifact using Manifest\n",
    "    else:\n",
    "        artifact = Artifact.import_data(d_type, manifest_file, view_type=v_type)\n",
    "        if not os.path.isfile(interim_demux_path):\n",
    "            artifact.save(interim_demux_path)\n",
    "            \n",
    "    # Filter and Save Artifact\n",
    "    artifact = filter_samples(demux=artifact, metadata=metadata_qa).filtered_demux\n",
    "    artifact.save(demux_file)\n",
    "    \n",
    "    # Visualization\n",
    "    if os.path.isfile(demux_view):\n",
    "        demux_view_obj = Visualization.load(demux_view)\n",
    "    else:\n",
    "        demux_view_obj = demux.visualizers.summarize(artifact).visualization\n",
    "        Visualization.save(demux_view_obj, filepath=demux_view)\n",
    "        \n",
    "elif os.path.isfile(demux_file):\n",
    "    \n",
    "    # Load Artifact\n",
    "    artifact = Artifact.load(demux_file)\n",
    "    artifact = filter_samples(demux=artifact, metadata=metadata_qa).filtered_demux\n",
    "    \n",
    "    # Visualization\n",
    "    if os.path.isfile(demux_view):\n",
    "        demux_view_obj = Visualization.load(demux_view)\n",
    "    else:\n",
    "        demux_view_obj = demux.visualizers.summarize(artifact).visualization\n",
    "        Visualization.save(demux_view_obj, filepath=demux_view)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6bccd614",
   "metadata": {
    "papermill": {
     "duration": 0.001808,
     "end_time": "2024-02-20T19:59:57.245511",
     "exception": false,
     "start_time": "2024-02-20T19:59:57.243703",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "## Step report\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "2fa6dd28",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-06-28T19:47:56.704841Z",
     "start_time": "2022-06-28T19:47:56.611846Z"
    },
    "execution": {
     "iopub.execute_input": "2024-02-20T19:59:57.249809Z",
     "iopub.status.busy": "2024-02-20T19:59:57.249679Z",
     "iopub.status.idle": "2024-02-20T19:59:57.254382Z",
     "shell.execute_reply": "2024-02-20T19:59:57.253863Z"
    },
    "papermill": {
     "duration": 0.00792,
     "end_time": "2024-02-20T19:59:57.255270",
     "exception": false,
     "start_time": "2024-02-20T19:59:57.247350",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<visualization: Visualization uuid: 1750180b-75bb-47b7-a998-194057e9a17d>\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/tmp/qiime2/lauro/data/1750180b-75bb-47b7-a998-194057e9a17d')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 1750180b-75bb-47b7-a998-194057e9a17d>"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(demux_view_obj)\n",
    "\n",
    "# Render Visualization\n",
    "demux_view_obj"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "61e1c5fd",
   "metadata": {
    "papermill": {
     "duration": 0.001909,
     "end_time": "2024-02-20T19:59:57.259074",
     "exception": false,
     "start_time": "2024-02-20T19:59:57.257165",
     "status": "completed"
    },
    "tags": []
   },
   "source": [
    "[cutadapt](https://docs.qiime2.org/2022.2/plugins/available/cutadapt/trim-paired/)\n",
    "\n",
    "We trim the forward primer and the reverse complement of the reverse primer from the forward reads. We trim the reverse primer and reverse complement of the forward primer from the reverse reads."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "b789290c",
   "metadata": {
    "execution": {
     "iopub.execute_input": "2024-02-20T19:59:57.263434Z",
     "iopub.status.busy": "2024-02-20T19:59:57.263314Z",
     "iopub.status.idle": "2024-02-20T20:03:37.517156Z",
     "shell.execute_reply": "2024-02-20T20:03:37.516638Z"
    },
    "papermill": {
     "duration": 220.257287,
     "end_time": "2024-02-20T20:03:37.518247",
     "exception": false,
     "start_time": "2024-02-20T19:59:57.260960",
     "status": "completed"
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV01_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV01_1_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV01_0_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV01_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV01_0_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV01_1_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV01_0_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV01_1_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.252 s (5.209 µs/read; 11.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,363\n",
      "  Read 1 with adapter:                  46,667 (96.5%)\n",
      "  Read 2 with adapter:                  44,020 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,840 (12.1%)\n",
      "Pairs written (passing filters):        42,523 (87.9%)\n",
      "\n",
      "Total basepairs processed:    24,665,130 bp\n",
      "  Read 1:    14,750,715 bp\n",
      "  Read 2:     9,914,415 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     19,823,881 bp (80.4%)\n",
      "  Read 1:    12,099,227 bp\n",
      "  Read 2:     7,724,654 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 46667 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t12\t0.0\t0\t12\n",
      "18\t7093\t0.0\t0\t7093\n",
      "19\t8356\t0.0\t0\t8356\n",
      "20\t8606\t0.0\t0\t8606\n",
      "21\t7832\t0.0\t0\t7832\n",
      "22\t7913\t0.0\t0\t7913\n",
      "23\t6842\t0.0\t0\t6842\n",
      "24\t5\t0.0\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 44020 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t18\t0.0\t0\t18\n",
      "21\t8261\t0.0\t0\t8261\n",
      "22\t8079\t0.0\t0\t8079\n",
      "23\t7308\t0.0\t0\t7308\n",
      "24\t7415\t0.0\t0\t7415\n",
      "25\t6521\t0.0\t0\t6521\n",
      "26\t6403\t0.0\t0\t6403\n",
      "27\t7\t0.0\t0\t7\n",
      "33\t1\t0.0\t0\t1\n",
      "43\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV02_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV02_3_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV02_2_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV02_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV02_2_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV02_3_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV02_2_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV02_3_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.155 s (7.786 µs/read; 7.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,951\n",
      "  Read 1 with adapter:                  19,173 (96.1%)\n",
      "  Read 2 with adapter:                  17,728 (88.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            2,898 (14.5%)\n",
      "Pairs written (passing filters):        17,053 (85.5%)\n",
      "\n",
      "Total basepairs processed:    10,175,010 bp\n",
      "  Read 1:     6,085,055 bp\n",
      "  Read 2:     4,089,955 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      7,950,024 bp (78.1%)\n",
      "  Read 1:     4,852,004 bp\n",
      "  Read 2:     3,098,020 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 19173 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t6\t0.0\t0\t6\n",
      "18\t2845\t0.0\t0\t2845\n",
      "19\t3510\t0.0\t0\t3510\n",
      "20\t3535\t0.0\t0\t3535\n",
      "21\t3147\t0.0\t0\t3147\n",
      "22\t3272\t0.0\t0\t3272\n",
      "23\t2853\t0.0\t0\t2853\n",
      "24\t3\t0.0\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17728 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t0\t2\n",
      "20\t10\t0.0\t0\t10\n",
      "21\t3372\t0.0\t0\t3372\n",
      "22\t3171\t0.0\t0\t3171\n",
      "23\t2992\t0.0\t0\t2992\n",
      "24\t3021\t0.0\t0\t3021\n",
      "25\t2634\t0.0\t0\t2634\n",
      "26\t2524\t0.0\t0\t2524\n",
      "27\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV03_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV03_5_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV03_4_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV03_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV03_4_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV03_5_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV03_4_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV03_5_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.174 s (7.994 µs/read; 7.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             21,790\n",
      "  Read 1 with adapter:                  20,955 (96.2%)\n",
      "  Read 2 with adapter:                  19,090 (87.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            3,418 (15.7%)\n",
      "Pairs written (passing filters):        18,372 (84.3%)\n",
      "\n",
      "Total basepairs processed:    11,112,900 bp\n",
      "  Read 1:     6,645,950 bp\n",
      "  Read 2:     4,466,950 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,564,679 bp (77.1%)\n",
      "  Read 1:     5,227,448 bp\n",
      "  Read 2:     3,337,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 20955 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t3223\t0.0\t0\t3223\n",
      "19\t3754\t0.0\t0\t3754\n",
      "20\t3864\t0.0\t0\t3864\n",
      "21\t3494\t0.0\t0\t3494\n",
      "22\t3601\t0.0\t0\t3601\n",
      "23\t3013\t0.0\t0\t3013\n",
      "24\t3\t0.0\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19090 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "20\t6\t0.0\t0\t6\n",
      "21\t3632\t0.0\t0\t3632\n",
      "22\t3387\t0.0\t0\t3387\n",
      "23\t3174\t0.0\t0\t3174\n",
      "24\t3208\t0.0\t0\t3208\n",
      "25\t2862\t0.0\t0\t2862\n",
      "26\t2814\t0.0\t0\t2814\n",
      "27\t3\t0.0\t0\t3\n",
      "47\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV04_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV04_7_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV04_6_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV04_7_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV04_6_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV04_7_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV04_6_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV04_7_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.092 s (3.392 µs/read; 17.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            321,913\n",
      "  Read 1 with adapter:                 310,496 (96.5%)\n",
      "  Read 2 with adapter:                 288,741 (89.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           43,149 (13.4%)\n",
      "Pairs written (passing filters):       278,764 (86.6%)\n",
      "\n",
      "Total basepairs processed:   164,175,630 bp\n",
      "  Read 1:    98,183,465 bp\n",
      "  Read 2:    65,992,165 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    129,955,949 bp (79.2%)\n",
      "  Read 1:    79,318,484 bp\n",
      "  Read 2:    50,637,465 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 310496 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t1.2\t0\t2\n",
      "10\t5\t0.3\t0\t5\n",
      "11\t4\t0.1\t0\t4\n",
      "12\t6\t0.0\t0\t6\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t22\t0.0\t0\t22\n",
      "15\t13\t0.0\t0\t13\n",
      "16\t15\t0.0\t0\t15\n",
      "17\t78\t0.0\t0\t78\n",
      "18\t47566\t0.0\t0\t47566\n",
      "19\t55472\t0.0\t0\t55472\n",
      "20\t57333\t0.0\t0\t57333\n",
      "21\t51738\t0.0\t0\t51738\n",
      "22\t52893\t0.0\t0\t52893\n",
      "23\t45309\t0.0\t0\t45309\n",
      "24\t33\t0.0\t0\t33\n",
      "25\t1\t0.0\t0\t1\n",
      "26\t1\t0.0\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 288741 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t3\t0.1\t0\t3\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t5\t0.0\t0\t5\n",
      "17\t6\t0.0\t0\t6\n",
      "18\t6\t0.0\t0\t6\n",
      "19\t9\t0.0\t0\t9\n",
      "20\t105\t0.0\t0\t105\n",
      "21\t54238\t0.0\t0\t54238\n",
      "22\t51743\t0.0\t0\t51743\n",
      "23\t48635\t0.0\t0\t48635\n",
      "24\t48510\t0.0\t0\t48510\n",
      "25\t43310\t0.0\t0\t43310\n",
      "26\t42108\t0.0\t0\t42108\n",
      "27\t37\t0.0\t0\t37\n",
      "28\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "63\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV05_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV05_9_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV05_8_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV05_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV05_8_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV05_9_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV05_8_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV05_9_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.849 s (3.635 µs/read; 16.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            233,637\n",
      "  Read 1 with adapter:                 225,243 (96.4%)\n",
      "  Read 2 with adapter:                 212,325 (90.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           28,719 (12.3%)\n",
      "Pairs written (passing filters):       204,918 (87.7%)\n",
      "\n",
      "Total basepairs processed:   119,154,870 bp\n",
      "  Read 1:    71,259,285 bp\n",
      "  Read 2:    47,895,585 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     95,527,115 bp (80.2%)\n",
      "  Read 1:    58,305,492 bp\n",
      "  Read 2:    37,221,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 225243 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t3.6\t0\t1\n",
      "9\t1\t0.9\t0\t1\n",
      "10\t3\t0.2\t0\t3\n",
      "11\t8\t0.1\t0\t8\n",
      "12\t4\t0.0\t0\t4\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t5\t0.0\t0\t5\n",
      "15\t10\t0.0\t0\t10\n",
      "16\t13\t0.0\t0\t13\n",
      "17\t65\t0.0\t0\t65\n",
      "18\t34116\t0.0\t0\t34116\n",
      "19\t40108\t0.0\t0\t40108\n",
      "20\t41910\t0.0\t0\t41910\n",
      "21\t37689\t0.0\t0\t37689\n",
      "22\t38677\t0.0\t0\t38677\n",
      "23\t32606\t0.0\t0\t32606\n",
      "24\t26\t0.0\t0\t26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 212325 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t5\t0.0\t0\t5\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t5\t0.0\t0\t5\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t4\t0.0\t0\t4\n",
      "20\t78\t0.0\t0\t78\n",
      "21\t39676\t0.0\t0\t39676\n",
      "22\t37904\t0.0\t0\t37904\n",
      "23\t35753\t0.0\t0\t35753\n",
      "24\t35561\t0.0\t0\t35561\n",
      "25\t32193\t0.0\t0\t32193\n",
      "26\t31115\t0.0\t0\t31115\n",
      "27\t20\t0.0\t0\t20\n",
      "30\t1\t0.0\t0\t1\n",
      "40\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "48\t3\t0.0\t0\t3\n",
      "52\t1\t0.0\t0\t1\n",
      "59\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV06_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV06_11_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV06_10_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV06_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV06_10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV06_11_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV06_10_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV06_11_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 2.026 s (3.336 µs/read; 17.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            607,471\n",
      "  Read 1 with adapter:                 586,758 (96.6%)\n",
      "  Read 2 with adapter:                 552,438 (90.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           73,316 (12.1%)\n",
      "Pairs written (passing filters):       534,155 (87.9%)\n",
      "\n",
      "Total basepairs processed:   309,810,210 bp\n",
      "  Read 1:   185,278,655 bp\n",
      "  Read 2:   124,531,555 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    249,018,751 bp (80.4%)\n",
      "  Read 1:   151,987,012 bp\n",
      "  Read 2:    97,031,739 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 586758 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t9.3\t0\t2\n",
      "9\t5\t2.3\t0\t5\n",
      "10\t4\t0.6\t0\t4\n",
      "11\t11\t0.1\t0\t11\n",
      "12\t6\t0.0\t0\t6\n",
      "13\t5\t0.0\t0\t5\n",
      "14\t25\t0.0\t0\t25\n",
      "15\t20\t0.0\t0\t20\n",
      "16\t32\t0.0\t0\t32\n",
      "17\t156\t0.0\t0\t156\n",
      "18\t90288\t0.0\t0\t90288\n",
      "19\t104580\t0.0\t0\t104580\n",
      "20\t108991\t0.0\t0\t108991\n",
      "21\t97081\t0.0\t0\t97081\n",
      "22\t100306\t0.0\t0\t100306\n",
      "23\t85186\t0.0\t0\t85186\n",
      "24\t52\t0.0\t0\t52\n",
      "26\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "43\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 552438 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t2.3\t0\t1\n",
      "10\t2\t0.6\t0\t2\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t5\t0.0\t0\t5\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t19\t0.0\t0\t19\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t7\t0.0\t0\t7\n",
      "18\t8\t0.0\t0\t8\n",
      "19\t19\t0.0\t0\t19\n",
      "20\t181\t0.0\t0\t181\n",
      "21\t103786\t0.0\t0\t103786\n",
      "22\t99240\t0.0\t0\t99240\n",
      "23\t93416\t0.0\t0\t93416\n",
      "24\t92709\t0.0\t0\t92709\n",
      "25\t83008\t0.0\t0\t83008\n",
      "26\t79955\t0.0\t0\t79955\n",
      "27\t63\t0.0\t0\t63\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t2\t0.0\t0\t2\n",
      "31\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "46\t3\t0.0\t0\t3\n",
      "48\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "60\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV07_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV07_13_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV07_12_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV07_13_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV07_12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV07_13_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV07_12_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV07_13_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.245 s (5.641 µs/read; 10.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,364\n",
      "  Read 1 with adapter:                  41,914 (96.7%)\n",
      "  Read 2 with adapter:                  39,373 (90.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,276 (12.2%)\n",
      "Pairs written (passing filters):        38,088 (87.8%)\n",
      "\n",
      "Total basepairs processed:    22,115,640 bp\n",
      "  Read 1:    13,226,020 bp\n",
      "  Read 2:     8,889,620 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     17,755,349 bp (80.3%)\n",
      "  Read 1:    10,837,192 bp\n",
      "  Read 2:     6,918,157 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 41914 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t6\t0.0\t0\t6\n",
      "18\t6432\t0.0\t0\t6432\n",
      "19\t7350\t0.0\t0\t7350\n",
      "20\t7772\t0.0\t0\t7772\n",
      "21\t7034\t0.0\t0\t7034\n",
      "22\t7165\t0.0\t0\t7165\n",
      "23\t6146\t0.0\t0\t6146\n",
      "24\t4\t0.0\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 39373 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t0\t1\n",
      "18\t5\t0.0\t0\t5\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t12\t0.0\t0\t12\n",
      "21\t7254\t0.0\t0\t7254\n",
      "22\t7085\t0.0\t0\t7085\n",
      "23\t6603\t0.0\t0\t6603\n",
      "24\t6701\t0.0\t0\t6701\n",
      "25\t5952\t0.0\t0\t5952\n",
      "26\t5753\t0.0\t0\t5753\n",
      "27\t4\t0.0\t0\t4\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV08_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV08_15_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV08_14_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV08_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV08_14_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV08_15_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV08_14_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV08_15_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.298 s (4.277 µs/read; 14.03 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,746\n",
      "  Read 1 with adapter:                  67,359 (96.6%)\n",
      "  Read 2 with adapter:                  63,498 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            8,348 (12.0%)\n",
      "Pairs written (passing filters):        61,398 (88.0%)\n",
      "\n",
      "Total basepairs processed:    35,570,460 bp\n",
      "  Read 1:    21,272,530 bp\n",
      "  Read 2:    14,297,930 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     28,623,338 bp (80.5%)\n",
      "  Read 1:    17,469,760 bp\n",
      "  Read 2:    11,153,578 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 67359 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.1\t0\t1\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t17\t0.0\t0\t17\n",
      "18\t10420\t0.0\t0\t10420\n",
      "19\t11878\t0.0\t0\t11878\n",
      "20\t12369\t0.0\t0\t12369\n",
      "21\t11268\t0.0\t0\t11268\n",
      "22\t11547\t0.0\t0\t11547\n",
      "23\t9842\t0.0\t0\t9842\n",
      "24\t7\t0.0\t0\t7\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 63498 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "20\t26\t0.0\t0\t26\n",
      "21\t12101\t0.0\t0\t12101\n",
      "22\t11389\t0.0\t0\t11389\n",
      "23\t10672\t0.0\t0\t10672\n",
      "24\t10632\t0.0\t0\t10632\n",
      "25\t9542\t0.0\t0\t9542\n",
      "26\t9120\t0.0\t0\t9120\n",
      "27\t8\t0.0\t0\t8\n",
      "38\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV10_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV10_17_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV10_16_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV10_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV10_16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV10_17_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV10_16_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV10_17_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.965 s (3.706 µs/read; 16.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            260,324\n",
      "  Read 1 with adapter:                 251,599 (96.6%)\n",
      "  Read 2 with adapter:                 237,700 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           30,351 (11.7%)\n",
      "Pairs written (passing filters):       229,973 (88.3%)\n",
      "\n",
      "Total basepairs processed:   132,765,240 bp\n",
      "  Read 1:    79,398,820 bp\n",
      "  Read 2:    53,366,420 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    107,210,223 bp (80.8%)\n",
      "  Read 1:    65,435,655 bp\n",
      "  Read 2:    41,774,568 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 251599 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t4.0\t0\t2\n",
      "9\t2\t1.0\t0\t2\n",
      "10\t2\t0.2\t0\t2\n",
      "11\t5\t0.1\t0\t5\n",
      "12\t6\t0.0\t0\t6\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t10\t0.0\t0\t10\n",
      "15\t14\t0.0\t0\t14\n",
      "16\t11\t0.0\t0\t11\n",
      "17\t77\t0.0\t0\t77\n",
      "18\t38367\t0.0\t0\t38367\n",
      "19\t45018\t0.0\t0\t45018\n",
      "20\t46668\t0.0\t0\t46668\n",
      "21\t42042\t0.0\t0\t42042\n",
      "22\t42829\t0.0\t0\t42829\n",
      "23\t36509\t0.0\t0\t36509\n",
      "24\t35\t0.0\t0\t35\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 237700 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t7\t0.0\t0\t7\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t2\t0.0\t0\t2\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t11\t0.0\t0\t11\n",
      "20\t72\t0.0\t0\t72\n",
      "21\t44567\t0.0\t0\t44567\n",
      "22\t42785\t0.0\t0\t42785\n",
      "23\t39826\t0.0\t0\t39826\n",
      "24\t40102\t0.0\t0\t40102\n",
      "25\t35865\t0.0\t0\t35865\n",
      "26\t34425\t0.0\t0\t34425\n",
      "27\t28\t0.0\t0\t28\n",
      "33\t1\t0.0\t0\t1\n",
      "40\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "56\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV102_120_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV102_121_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV102_120_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV102_121_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV102_120_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV102_121_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV102_120_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV102_121_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.222 s (5.929 µs/read; 10.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,438\n",
      "  Read 1 with adapter:                  36,105 (96.4%)\n",
      "  Read 2 with adapter:                  33,265 (88.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,302 (14.2%)\n",
      "Pairs written (passing filters):        32,136 (85.8%)\n",
      "\n",
      "Total basepairs processed:    19,093,380 bp\n",
      "  Read 1:    11,418,590 bp\n",
      "  Read 2:     7,674,790 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     14,981,903 bp (78.5%)\n",
      "  Read 1:     9,144,877 bp\n",
      "  Read 2:     5,837,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 36105 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t2\t0.0\t0\t2\n",
      "17\t8\t0.0\t0\t8\n",
      "18\t5704\t0.0\t0\t5704\n",
      "19\t6542\t0.0\t0\t6542\n",
      "20\t6562\t0.0\t0\t6562\n",
      "21\t5931\t0.0\t0\t5931\n",
      "22\t6070\t0.0\t0\t6070\n",
      "23\t5277\t0.0\t0\t5277\n",
      "24\t6\t0.0\t0\t6\n",
      "28\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 33265 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "20\t8\t0.0\t0\t8\n",
      "21\t6048\t0.0\t0\t6048\n",
      "22\t6085\t0.0\t0\t6085\n",
      "23\t5608\t0.0\t0\t5608\n",
      "24\t5602\t0.0\t0\t5602\n",
      "25\t5121\t0.0\t0\t5121\n",
      "26\t4783\t0.0\t0\t4783\n",
      "27\t2\t0.0\t0\t2\n",
      "47\t2\t0.0\t0\t2\n",
      "49\t1\t0.0\t0\t1\n",
      "51\t2\t0.0\t0\t2\n",
      "52\t1\t0.0\t0\t1\n",
      "75\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV103_122_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV103_123_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV103_122_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV103_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV103_122_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV103_123_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV103_122_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV103_123_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.255 s (5.415 µs/read; 11.08 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,145\n",
      "  Read 1 with adapter:                  45,512 (96.5%)\n",
      "  Read 2 with adapter:                  42,827 (90.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,742 (12.2%)\n",
      "Pairs written (passing filters):        41,403 (87.8%)\n",
      "\n",
      "Total basepairs processed:    24,043,950 bp\n",
      "  Read 1:    14,379,225 bp\n",
      "  Read 2:     9,664,725 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     19,303,074 bp (80.3%)\n",
      "  Read 1:    11,782,124 bp\n",
      "  Read 2:     7,520,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 45512 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.0\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t11\t0.0\t0\t11\n",
      "18\t7307\t0.0\t0\t7307\n",
      "19\t8163\t0.0\t0\t8163\n",
      "20\t8419\t0.0\t0\t8419\n",
      "21\t7455\t0.0\t0\t7455\n",
      "22\t7608\t0.0\t0\t7608\n",
      "23\t6536\t0.0\t0\t6536\n",
      "24\t6\t0.0\t0\t6\n",
      "54\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 42827 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t7\t0.0\t0\t7\n",
      "21\t7994\t0.0\t0\t7994\n",
      "22\t7743\t0.0\t0\t7743\n",
      "23\t7241\t0.0\t0\t7241\n",
      "24\t7247\t0.0\t0\t7247\n",
      "25\t6446\t0.0\t0\t6446\n",
      "26\t6139\t0.0\t0\t6139\n",
      "27\t5\t0.0\t0\t5\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV104_124_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV104_125_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV104_124_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV104_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV104_124_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV104_125_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV104_124_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV104_125_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.325 s (4.895 µs/read; 12.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,331\n",
      "  Read 1 with adapter:                  64,072 (96.6%)\n",
      "  Read 2 with adapter:                  60,445 (91.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            7,881 (11.9%)\n",
      "Pairs written (passing filters):        58,450 (88.1%)\n",
      "\n",
      "Total basepairs processed:    33,828,810 bp\n",
      "  Read 1:    20,230,955 bp\n",
      "  Read 2:    13,597,855 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     27,250,743 bp (80.6%)\n",
      "  Read 1:    16,633,394 bp\n",
      "  Read 2:    10,617,349 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 64072 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.0\t0\t1\n",
      "9\t2\t0.3\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t5\t0.0\t0\t5\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t4\t0.0\t0\t4\n",
      "17\t12\t0.0\t0\t12\n",
      "18\t10299\t0.0\t0\t10299\n",
      "19\t11485\t0.0\t0\t11485\n",
      "20\t11757\t0.0\t0\t11757\n",
      "21\t10741\t0.0\t0\t10741\n",
      "22\t10683\t0.0\t0\t10683\n",
      "23\t9069\t0.0\t0\t9069\n",
      "24\t8\t0.0\t0\t8\n",
      "36\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 60445 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t2\t0.0\t0\t2\n",
      "17\t1\t0.0\t0\t1\n",
      "20\t22\t0.0\t0\t22\n",
      "21\t11212\t0.0\t0\t11212\n",
      "22\t10931\t0.0\t0\t10931\n",
      "23\t10156\t0.0\t0\t10156\n",
      "24\t10223\t0.0\t0\t10223\n",
      "25\t9154\t0.0\t0\t9154\n",
      "26\t8737\t0.0\t0\t8737\n",
      "27\t5\t0.0\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV107_126_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV107_127_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV107_126_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV107_127_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV107_126_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV107_127_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV107_126_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV107_127_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.707 s (3.243 µs/read; 18.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            218,138\n",
      "  Read 1 with adapter:                 211,017 (96.7%)\n",
      "  Read 2 with adapter:                 195,200 (89.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           29,031 (13.3%)\n",
      "Pairs written (passing filters):       189,107 (86.7%)\n",
      "\n",
      "Total basepairs processed:   111,250,380 bp\n",
      "  Read 1:    66,532,090 bp\n",
      "  Read 2:    44,718,290 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     88,166,310 bp (79.3%)\n",
      "  Read 1:    53,816,111 bp\n",
      "  Read 2:    34,350,199 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 211017 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t3\t3.3\t0\t3\n",
      "9\t1\t0.8\t0\t1\n",
      "10\t3\t0.2\t0\t3\n",
      "11\t5\t0.1\t0\t5\n",
      "12\t6\t0.0\t0\t6\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t9\t0.0\t0\t9\n",
      "15\t5\t0.0\t0\t5\n",
      "16\t10\t0.0\t0\t10\n",
      "17\t63\t0.0\t0\t63\n",
      "18\t34168\t0.0\t0\t34168\n",
      "19\t37980\t0.0\t0\t37980\n",
      "20\t38516\t0.0\t0\t38516\n",
      "21\t34953\t0.0\t0\t34953\n",
      "22\t35524\t0.0\t0\t35524\n",
      "23\t29743\t0.0\t0\t29743\n",
      "24\t27\t0.0\t0\t27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 195200 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.8\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t6\t0.0\t0\t6\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t7\t0.0\t0\t7\n",
      "18\t6\t0.0\t0\t6\n",
      "19\t11\t0.0\t0\t11\n",
      "20\t67\t0.0\t0\t67\n",
      "21\t36213\t0.0\t0\t36213\n",
      "22\t35325\t0.0\t0\t35325\n",
      "23\t32587\t0.0\t0\t32587\n",
      "24\t33038\t0.0\t0\t33038\n",
      "25\t29525\t0.0\t0\t29525\n",
      "26\t28376\t0.0\t0\t28376\n",
      "27\t25\t0.0\t0\t25\n",
      "31\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "52\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV109_128_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV109_129_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV109_128_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV109_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV109_128_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV109_129_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV109_128_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV109_129_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.262 s (5.185 µs/read; 11.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,448\n",
      "  Read 1 with adapter:                  48,769 (96.7%)\n",
      "  Read 2 with adapter:                  45,129 (89.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            6,775 (13.4%)\n",
      "Pairs written (passing filters):        43,673 (86.6%)\n",
      "\n",
      "Total basepairs processed:    25,728,480 bp\n",
      "  Read 1:    15,386,640 bp\n",
      "  Read 2:    10,341,840 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     20,359,999 bp (79.1%)\n",
      "  Read 1:    12,426,741 bp\n",
      "  Read 2:     7,933,258 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 48769 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.8\t0\t1\n",
      "10\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t23\t0.0\t0\t23\n",
      "18\t7686\t0.0\t0\t7686\n",
      "19\t8559\t0.0\t0\t8559\n",
      "20\t8800\t0.0\t0\t8800\n",
      "21\t8274\t0.0\t0\t8274\n",
      "22\t8244\t0.0\t0\t8244\n",
      "23\t7169\t0.0\t0\t7169\n",
      "24\t3\t0.0\t0\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 45129 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t2\t0.0\t0\t2\n",
      "20\t17\t0.0\t0\t17\n",
      "21\t8509\t0.0\t0\t8509\n",
      "22\t8272\t0.0\t0\t8272\n",
      "23\t7321\t0.0\t0\t7321\n",
      "24\t7577\t0.0\t0\t7577\n",
      "25\t6750\t0.0\t0\t6750\n",
      "26\t6674\t0.0\t0\t6674\n",
      "27\t5\t0.0\t0\t5\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV11_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV11_19_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV11_18_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV11_19_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV11_18_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV11_19_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV11_18_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV11_19_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.690 s (3.739 µs/read; 16.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            184,403\n",
      "  Read 1 with adapter:                 177,445 (96.2%)\n",
      "  Read 2 with adapter:                 167,265 (90.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           23,248 (12.6%)\n",
      "Pairs written (passing filters):       161,155 (87.4%)\n",
      "\n",
      "Total basepairs processed:    94,045,530 bp\n",
      "  Read 1:    56,242,915 bp\n",
      "  Read 2:    37,802,615 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     75,127,431 bp (79.9%)\n",
      "  Read 1:    45,854,994 bp\n",
      "  Read 2:    29,272,437 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 177445 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t4\t2.8\t0\t4\n",
      "9\t1\t0.7\t0\t1\n",
      "11\t3\t0.0\t0\t3\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t12\t0.0\t0\t12\n",
      "16\t9\t0.0\t0\t9\n",
      "17\t43\t0.0\t0\t43\n",
      "18\t27276\t0.0\t0\t27276\n",
      "19\t31702\t0.0\t0\t31702\n",
      "20\t32798\t0.0\t0\t32798\n",
      "21\t29273\t0.0\t0\t29273\n",
      "22\t30469\t0.0\t0\t30469\n",
      "23\t25829\t0.0\t0\t25829\n",
      "24\t16\t0.0\t0\t16\n",
      "25\t1\t0.0\t0\t1\n",
      "28\t1\t0.0\t0\t1\n",
      "42\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 167265 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t9\t0.0\t0\t9\n",
      "20\t60\t0.0\t0\t60\n",
      "21\t31195\t0.0\t0\t31195\n",
      "22\t30017\t0.0\t0\t30017\n",
      "23\t28164\t0.0\t0\t28164\n",
      "24\t28003\t0.0\t0\t28003\n",
      "25\t25190\t0.0\t0\t25190\n",
      "26\t24587\t0.0\t0\t24587\n",
      "27\t24\t0.0\t0\t24\n",
      "30\t2\t0.0\t0\t2\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t1\t0.0\t0\t1\n",
      "51\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "61\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV110_130_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV110_131_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV110_130_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV110_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV110_130_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV110_131_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV110_130_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV110_131_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.452 s (4.570 µs/read; 13.13 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             98,807\n",
      "  Read 1 with adapter:                  95,554 (96.7%)\n",
      "  Read 2 with adapter:                  90,074 (91.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           11,629 (11.8%)\n",
      "Pairs written (passing filters):        87,178 (88.2%)\n",
      "\n",
      "Total basepairs processed:    50,391,570 bp\n",
      "  Read 1:    30,136,135 bp\n",
      "  Read 2:    20,255,435 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     40,643,386 bp (80.7%)\n",
      "  Read 1:    24,808,467 bp\n",
      "  Read 2:    15,834,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 95554 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.5\t0\t1\n",
      "10\t2\t0.1\t0\t2\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t6\t0.0\t0\t6\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t6\t0.0\t0\t6\n",
      "17\t24\t0.0\t0\t24\n",
      "18\t15458\t0.0\t0\t15458\n",
      "19\t17135\t0.0\t0\t17135\n",
      "20\t17285\t0.0\t0\t17285\n",
      "21\t15994\t0.0\t0\t15994\n",
      "22\t16007\t0.0\t0\t16007\n",
      "23\t13616\t0.0\t0\t13616\n",
      "24\t11\t0.0\t0\t11\n",
      "27\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 90074 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t2\t0.0\t0\t2\n",
      "19\t5\t0.0\t0\t5\n",
      "20\t26\t0.0\t0\t26\n",
      "21\t16705\t0.0\t0\t16705\n",
      "22\t16100\t0.0\t0\t16100\n",
      "23\t15013\t0.0\t0\t15013\n",
      "24\t15314\t0.0\t0\t15314\n",
      "25\t13845\t0.0\t0\t13845\n",
      "26\t13044\t0.0\t0\t13044\n",
      "27\t11\t0.0\t0\t11\n",
      "49\t1\t0.0\t0\t1\n",
      "51\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV112_132_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV112_133_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV112_132_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV112_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV112_132_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV112_133_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV112_132_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV112_133_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.463 s (4.115 µs/read; 14.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            112,450\n",
      "  Read 1 with adapter:                 108,836 (96.8%)\n",
      "  Read 2 with adapter:                 103,451 (92.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           12,229 (10.9%)\n",
      "Pairs written (passing filters):       100,221 (89.1%)\n",
      "\n",
      "Total basepairs processed:    57,349,500 bp\n",
      "  Read 1:    34,297,250 bp\n",
      "  Read 2:    23,052,250 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     46,725,918 bp (81.5%)\n",
      "  Read 1:    28,521,088 bp\n",
      "  Read 2:    18,204,830 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 108836 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.4\t0\t3\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t9\t0.0\t0\t9\n",
      "15\t7\t0.0\t0\t7\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t19\t0.0\t0\t19\n",
      "18\t17666\t0.0\t0\t17666\n",
      "19\t19861\t0.0\t0\t19861\n",
      "20\t19575\t0.0\t0\t19575\n",
      "21\t18039\t0.0\t0\t18039\n",
      "22\t18294\t0.0\t0\t18294\n",
      "23\t15331\t0.0\t0\t15331\n",
      "24\t14\t0.0\t0\t14\n",
      "25\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103451 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t0\t3\n",
      "15\t4\t0.0\t0\t4\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t4\t0.0\t0\t4\n",
      "20\t37\t0.0\t0\t37\n",
      "21\t19237\t0.0\t0\t19237\n",
      "22\t18691\t0.0\t0\t18691\n",
      "23\t17318\t0.0\t0\t17318\n",
      "24\t17493\t0.0\t0\t17493\n",
      "25\t15627\t0.0\t0\t15627\n",
      "26\t15022\t0.0\t0\t15022\n",
      "27\t11\t0.0\t0\t11\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV113_134_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV113_135_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV113_134_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV113_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV113_134_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV113_135_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV113_134_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV113_135_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.703 s (3.618 µs/read; 16.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            194,398\n",
      "  Read 1 with adapter:                 187,827 (96.6%)\n",
      "  Read 2 with adapter:                 176,997 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           23,165 (11.9%)\n",
      "Pairs written (passing filters):       171,233 (88.1%)\n",
      "\n",
      "Total basepairs processed:    99,142,980 bp\n",
      "  Read 1:    59,291,390 bp\n",
      "  Read 2:    39,851,590 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     79,831,463 bp (80.5%)\n",
      "  Read 1:    48,728,431 bp\n",
      "  Read 2:    31,103,032 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 187827 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t3.0\t0\t2\n",
      "9\t2\t0.7\t0\t2\n",
      "10\t2\t0.2\t0\t2\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t9\t0.0\t0\t9\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t6\t0.0\t0\t6\n",
      "17\t59\t0.0\t0\t59\n",
      "18\t30193\t0.0\t0\t30193\n",
      "19\t33553\t0.0\t0\t33553\n",
      "20\t34608\t0.0\t0\t34608\n",
      "21\t31202\t0.0\t0\t31202\n",
      "22\t31669\t0.0\t0\t31669\n",
      "23\t26496\t0.0\t0\t26496\n",
      "24\t16\t0.0\t0\t16\n",
      "25\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 176997 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t3\t0.0\t0\t3\n",
      "17\t4\t0.0\t0\t4\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t8\t0.0\t0\t8\n",
      "20\t67\t0.0\t0\t67\n",
      "21\t33022\t0.0\t0\t33022\n",
      "22\t31651\t0.0\t0\t31651\n",
      "23\t29477\t0.0\t0\t29477\n",
      "24\t29982\t0.0\t0\t29982\n",
      "25\t26991\t0.0\t0\t26991\n",
      "26\t25748\t0.0\t0\t25748\n",
      "27\t28\t0.0\t0\t28\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t3\t0.0\t0\t3\n",
      "50\t1\t0.0\t0\t1\n",
      "52\t2\t0.0\t0\t2\n",
      "78\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV117_136_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV117_137_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV117_136_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV117_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV117_136_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV117_137_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV117_136_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV117_137_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.926 s (3.383 µs/read; 17.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            273,880\n",
      "  Read 1 with adapter:                 265,316 (96.9%)\n",
      "  Read 2 with adapter:                 249,935 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           31,506 (11.5%)\n",
      "Pairs written (passing filters):       242,374 (88.5%)\n",
      "\n",
      "Total basepairs processed:   139,678,800 bp\n",
      "  Read 1:    83,533,400 bp\n",
      "  Read 2:    56,145,400 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    113,000,818 bp (80.9%)\n",
      "  Read 1:    68,973,461 bp\n",
      "  Read 2:    44,027,357 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 265316 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t4.2\t0\t2\n",
      "9\t1\t1.0\t0\t1\n",
      "10\t3\t0.3\t0\t3\n",
      "11\t7\t0.1\t0\t7\n",
      "12\t4\t0.0\t0\t4\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t13\t0.0\t0\t13\n",
      "15\t19\t0.0\t0\t19\n",
      "16\t17\t0.0\t0\t17\n",
      "17\t65\t0.0\t0\t65\n",
      "18\t42787\t0.0\t0\t42787\n",
      "19\t47726\t0.0\t0\t47726\n",
      "20\t48701\t0.0\t0\t48701\n",
      "21\t43737\t0.0\t0\t43737\n",
      "22\t44415\t0.0\t0\t44415\n",
      "23\t37785\t0.0\t0\t37785\n",
      "24\t27\t0.0\t0\t27\n",
      "25\t1\t0.0\t0\t1\n",
      "26\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 249935 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t3\t0.0\t0\t3\n",
      "19\t9\t0.0\t0\t9\n",
      "20\t88\t0.0\t0\t88\n",
      "21\t46609\t0.0\t0\t46609\n",
      "22\t45132\t0.0\t0\t45132\n",
      "23\t41858\t0.0\t0\t41858\n",
      "24\t42199\t0.0\t0\t42199\n",
      "25\t37911\t0.0\t0\t37911\n",
      "26\t36089\t0.0\t0\t36089\n",
      "27\t20\t0.0\t0\t20\n",
      "62\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV119_138_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV119_139_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV119_138_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV119_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV119_138_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV119_139_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV119_138_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV119_139_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.042 s (3.968 µs/read; 15.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            262,512\n",
      "  Read 1 with adapter:                 254,312 (96.9%)\n",
      "  Read 2 with adapter:                 237,627 (90.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           32,077 (12.2%)\n",
      "Pairs written (passing filters):       230,435 (87.8%)\n",
      "\n",
      "Total basepairs processed:   133,881,120 bp\n",
      "  Read 1:    80,066,160 bp\n",
      "  Read 2:    53,814,960 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    107,434,494 bp (80.2%)\n",
      "  Read 1:    65,576,802 bp\n",
      "  Read 2:    41,857,692 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 254312 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t4.0\t0\t1\n",
      "9\t2\t1.0\t0\t2\n",
      "10\t1\t0.3\t0\t1\n",
      "11\t6\t0.1\t0\t6\n",
      "12\t6\t0.0\t0\t6\n",
      "13\t5\t0.0\t0\t5\n",
      "14\t12\t0.0\t0\t12\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t14\t0.0\t0\t14\n",
      "17\t70\t0.0\t0\t70\n",
      "18\t41174\t0.0\t0\t41174\n",
      "19\t46028\t0.0\t0\t46028\n",
      "20\t46588\t0.0\t0\t46588\n",
      "21\t41803\t0.0\t0\t41803\n",
      "22\t42287\t0.0\t0\t42287\n",
      "23\t36272\t0.0\t0\t36272\n",
      "24\t30\t0.0\t0\t30\n",
      "26\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "42\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 237627 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t2\t0.0\t0\t2\n",
      "15\t9\t0.0\t0\t9\n",
      "16\t5\t0.0\t0\t5\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t4\t0.0\t0\t4\n",
      "19\t6\t0.0\t0\t6\n",
      "20\t68\t0.0\t0\t68\n",
      "21\t44379\t0.0\t0\t44379\n",
      "22\t42748\t0.0\t0\t42748\n",
      "23\t39654\t0.0\t0\t39654\n",
      "24\t40107\t0.0\t0\t40107\n",
      "25\t36124\t0.0\t0\t36124\n",
      "26\t34470\t0.0\t0\t34470\n",
      "27\t36\t0.0\t0\t36\n",
      "30\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t2\t0.0\t0\t2\n",
      "51\t1\t0.0\t0\t1\n",
      "52\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV120_140_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV120_141_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV120_140_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV120_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV120_140_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV120_141_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV120_140_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV120_141_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.170 s (3.260 µs/read; 18.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            358,904\n",
      "  Read 1 with adapter:                 347,503 (96.8%)\n",
      "  Read 2 with adapter:                 323,284 (90.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           45,453 (12.7%)\n",
      "Pairs written (passing filters):       313,451 (87.3%)\n",
      "\n",
      "Total basepairs processed:   183,041,040 bp\n",
      "  Read 1:   109,465,720 bp\n",
      "  Read 2:    73,575,320 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    146,135,237 bp (79.8%)\n",
      "  Read 1:    89,197,442 bp\n",
      "  Read 2:    56,937,795 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 347503 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t5\t5.5\t0\t5\n",
      "9\t4\t1.4\t0\t4\n",
      "10\t7\t0.3\t0\t7\n",
      "11\t1\t0.1\t0\t1\n",
      "12\t4\t0.0\t0\t4\n",
      "13\t4\t0.0\t0\t4\n",
      "14\t21\t0.0\t0\t21\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t16\t0.0\t0\t16\n",
      "17\t111\t0.0\t0\t111\n",
      "18\t55522\t0.0\t0\t55522\n",
      "19\t62211\t0.0\t0\t62211\n",
      "20\t63561\t0.0\t0\t63561\n",
      "21\t57778\t0.0\t0\t57778\n",
      "22\t58405\t0.0\t0\t58405\n",
      "23\t49807\t0.0\t0\t49807\n",
      "24\t32\t0.0\t0\t32\n",
      "25\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 323284 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.3\t0\t1\n",
      "11\t2\t0.1\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "15\t12\t0.0\t0\t12\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t6\t0.0\t0\t6\n",
      "18\t5\t0.0\t0\t5\n",
      "19\t4\t0.0\t0\t4\n",
      "20\t122\t0.0\t0\t122\n",
      "21\t60866\t0.0\t0\t60866\n",
      "22\t57782\t0.0\t0\t57782\n",
      "23\t53950\t0.0\t0\t53950\n",
      "24\t54605\t0.0\t0\t54605\n",
      "25\t48790\t0.0\t0\t48790\n",
      "26\t47076\t0.0\t0\t47076\n",
      "27\t51\t0.0\t0\t51\n",
      "28\t1\t0.0\t0\t1\n",
      "44\t2\t0.0\t0\t2\n",
      "47\t3\t0.0\t0\t3\n",
      "51\t1\t0.0\t0\t1\n",
      "52\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV124_142_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV124_143_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV124_142_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV124_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV124_142_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV124_143_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV124_142_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV124_143_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.483 s (5.069 µs/read; 11.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             95,313\n",
      "  Read 1 with adapter:                  91,870 (96.4%)\n",
      "  Read 2 with adapter:                  84,364 (88.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           13,849 (14.5%)\n",
      "Pairs written (passing filters):        81,464 (85.5%)\n",
      "\n",
      "Total basepairs processed:    48,609,630 bp\n",
      "  Read 1:    29,070,465 bp\n",
      "  Read 2:    19,539,165 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     37,978,502 bp (78.1%)\n",
      "  Read 1:    23,182,713 bp\n",
      "  Read 2:    14,795,789 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 91870 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t2\t0.1\t0\t2\n",
      "11\t3\t0.0\t0\t3\n",
      "12\t2\t0.0\t0\t2\n",
      "14\t7\t0.0\t0\t7\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t23\t0.0\t0\t23\n",
      "18\t14697\t0.0\t0\t14697\n",
      "19\t16362\t0.0\t0\t16362\n",
      "20\t16697\t0.0\t0\t16697\n",
      "21\t15390\t0.0\t0\t15390\n",
      "22\t15568\t0.0\t0\t15568\n",
      "23\t13100\t0.0\t0\t13100\n",
      "24\t5\t0.0\t0\t5\n",
      "27\t1\t0.0\t0\t1\n",
      "40\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84364 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t20\t0.0\t0\t20\n",
      "21\t15337\t0.0\t0\t15337\n",
      "22\t15068\t0.0\t0\t15068\n",
      "23\t14310\t0.0\t0\t14310\n",
      "24\t14281\t0.0\t0\t14281\n",
      "25\t12874\t0.0\t0\t12874\n",
      "26\t12450\t0.0\t0\t12450\n",
      "27\t6\t0.0\t0\t6\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t3\t0.0\t0\t3\n",
      "49\t3\t0.0\t0\t3\n",
      "50\t2\t0.0\t0\t2\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV125_144_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV125_145_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV125_144_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV125_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV125_144_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV125_145_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV125_144_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV125_145_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.485 s (4.396 µs/read; 13.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            110,270\n",
      "  Read 1 with adapter:                 106,523 (96.6%)\n",
      "  Read 2 with adapter:                 100,561 (91.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           13,030 (11.8%)\n",
      "Pairs written (passing filters):        97,240 (88.2%)\n",
      "\n",
      "Total basepairs processed:    56,237,700 bp\n",
      "  Read 1:    33,632,350 bp\n",
      "  Read 2:    22,605,350 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     45,332,983 bp (80.6%)\n",
      "  Read 1:    27,671,091 bp\n",
      "  Read 2:    17,661,892 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 106523 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t1\t0.1\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t9\t0.0\t0\t9\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t6\t0.0\t0\t6\n",
      "17\t27\t0.0\t0\t27\n",
      "18\t16890\t0.0\t0\t16890\n",
      "19\t19256\t0.0\t0\t19256\n",
      "20\t19441\t0.0\t0\t19441\n",
      "21\t17640\t0.0\t0\t17640\n",
      "22\t17940\t0.0\t0\t17940\n",
      "23\t15290\t0.0\t0\t15290\n",
      "24\t10\t0.0\t0\t10\n",
      "36\t1\t0.0\t0\t1\n",
      "40\t1\t0.0\t0\t1\n",
      "41\t1\t0.0\t0\t1\n",
      "42\t1\t0.0\t0\t1\n",
      "58\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 100561 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "15\t6\t0.0\t0\t6\n",
      "17\t2\t0.0\t0\t2\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t8\t0.0\t0\t8\n",
      "20\t39\t0.0\t0\t39\n",
      "21\t18834\t0.0\t0\t18834\n",
      "22\t17937\t0.0\t0\t17937\n",
      "23\t16819\t0.0\t0\t16819\n",
      "24\t16944\t0.0\t0\t16944\n",
      "25\t14977\t0.0\t0\t14977\n",
      "26\t14939\t0.0\t0\t14939\n",
      "27\t14\t0.0\t0\t14\n",
      "44\t1\t0.0\t0\t1\n",
      "46\t2\t0.0\t0\t2\n",
      "47\t4\t0.0\t0\t4\n",
      "48\t10\t0.0\t0\t10\n",
      "49\t5\t0.0\t0\t5\n",
      "50\t5\t0.0\t0\t5\n",
      "51\t6\t0.0\t0\t6\n",
      "52\t6\t0.0\t0\t6\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV126_146_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV126_147_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV126_146_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV126_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV126_146_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV126_147_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV126_146_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV126_147_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.351 s (4.131 µs/read; 14.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,929\n",
      "  Read 1 with adapter:                  82,107 (96.7%)\n",
      "  Read 2 with adapter:                  76,369 (89.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           11,006 (13.0%)\n",
      "Pairs written (passing filters):        73,923 (87.0%)\n",
      "\n",
      "Total basepairs processed:    43,313,790 bp\n",
      "  Read 1:    25,903,345 bp\n",
      "  Read 2:    17,410,445 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     34,464,512 bp (79.6%)\n",
      "  Read 1:    21,036,530 bp\n",
      "  Read 2:    13,427,982 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 82107 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.1\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t4\t0.0\t0\t4\n",
      "17\t21\t0.0\t0\t21\n",
      "18\t13119\t0.0\t0\t13119\n",
      "19\t14913\t0.0\t0\t14913\n",
      "20\t15029\t0.0\t0\t15029\n",
      "21\t13374\t0.0\t0\t13374\n",
      "22\t13949\t0.0\t0\t13949\n",
      "23\t11678\t0.0\t0\t11678\n",
      "24\t10\t0.0\t0\t10\n",
      "25\t1\t0.0\t0\t1\n",
      "62\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 76369 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t26\t0.0\t0\t26\n",
      "21\t14358\t0.0\t0\t14358\n",
      "22\t13848\t0.0\t0\t13848\n",
      "23\t12589\t0.0\t0\t12589\n",
      "24\t12836\t0.0\t0\t12836\n",
      "25\t11633\t0.0\t0\t11633\n",
      "26\t11060\t0.0\t0\t11060\n",
      "27\t7\t0.0\t0\t7\n",
      "53\t1\t0.0\t0\t1\n",
      "76\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV13_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV13_21_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV13_20_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV13_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV13_20_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV13_21_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV13_20_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV13_21_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.327 s (5.332 µs/read; 11.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,311\n",
      "  Read 1 with adapter:                  59,011 (96.2%)\n",
      "  Read 2 with adapter:                  55,340 (90.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            7,992 (13.0%)\n",
      "Pairs written (passing filters):        53,319 (87.0%)\n",
      "\n",
      "Total basepairs processed:    31,268,610 bp\n",
      "  Read 1:    18,699,855 bp\n",
      "  Read 2:    12,568,755 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     24,856,417 bp (79.5%)\n",
      "  Read 1:    15,171,199 bp\n",
      "  Read 2:     9,685,218 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 59011 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t3\t0.0\t0\t3\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t13\t0.0\t0\t13\n",
      "18\t9078\t0.0\t0\t9078\n",
      "19\t10542\t0.0\t0\t10542\n",
      "20\t10925\t0.0\t0\t10925\n",
      "21\t9630\t0.0\t0\t9630\n",
      "22\t9990\t0.0\t0\t9990\n",
      "23\t8812\t0.0\t0\t8812\n",
      "24\t12\t0.0\t0\t12\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 55340 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t16\t0.0\t0\t16\n",
      "21\t10368\t0.0\t0\t10368\n",
      "22\t9962\t0.0\t0\t9962\n",
      "23\t9323\t0.0\t0\t9323\n",
      "24\t9211\t0.0\t0\t9211\n",
      "25\t8366\t0.0\t0\t8366\n",
      "26\t8076\t0.0\t0\t8076\n",
      "27\t4\t0.0\t0\t4\n",
      "46\t2\t0.0\t0\t2\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV131_148_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV131_149_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV131_148_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV131_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV131_148_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV131_149_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV131_148_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV131_149_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.491 s (4.228 µs/read; 14.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            116,049\n",
      "  Read 1 with adapter:                 112,163 (96.7%)\n",
      "  Read 2 with adapter:                 105,171 (90.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           14,260 (12.3%)\n",
      "Pairs written (passing filters):       101,789 (87.7%)\n",
      "\n",
      "Total basepairs processed:    59,184,990 bp\n",
      "  Read 1:    35,394,945 bp\n",
      "  Read 2:    23,790,045 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     47,456,490 bp (80.2%)\n",
      "  Read 1:    28,966,774 bp\n",
      "  Read 2:    18,489,716 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 112163 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.8\t0\t1\n",
      "10\t5\t0.1\t0\t5\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t3\t0.0\t0\t3\n",
      "14\t6\t0.0\t0\t6\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t36\t0.0\t0\t36\n",
      "18\t18092\t0.0\t0\t18092\n",
      "19\t20169\t0.0\t0\t20169\n",
      "20\t20647\t0.0\t0\t20647\n",
      "21\t18389\t0.0\t0\t18389\n",
      "22\t18675\t0.0\t0\t18675\n",
      "23\t16118\t0.0\t0\t16118\n",
      "24\t13\t0.0\t0\t13\n",
      "26\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 105171 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t6\t0.0\t0\t6\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t47\t0.0\t0\t47\n",
      "21\t19605\t0.0\t0\t19605\n",
      "22\t19098\t0.0\t0\t19098\n",
      "23\t17614\t0.0\t0\t17614\n",
      "24\t17438\t0.0\t0\t17438\n",
      "25\t15974\t0.0\t0\t15974\n",
      "26\t15366\t0.0\t0\t15366\n",
      "27\t12\t0.0\t0\t12\n",
      "39\t1\t0.0\t0\t1\n",
      "49\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV133_150_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV133_151_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV133_150_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV133_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV133_150_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV133_151_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV133_150_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV133_151_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.439 s (4.347 µs/read; 13.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            101,042\n",
      "  Read 1 with adapter:                  97,718 (96.7%)\n",
      "  Read 2 with adapter:                  91,926 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           12,027 (11.9%)\n",
      "Pairs written (passing filters):        89,015 (88.1%)\n",
      "\n",
      "Total basepairs processed:    51,531,420 bp\n",
      "  Read 1:    30,817,810 bp\n",
      "  Read 2:    20,713,610 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     41,499,315 bp (80.5%)\n",
      "  Read 1:    25,330,890 bp\n",
      "  Read 2:    16,168,425 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 97718 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t5\t0.0\t0\t5\n",
      "12\t2\t0.0\t0\t2\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t5\t0.0\t0\t5\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t27\t0.0\t0\t27\n",
      "18\t15737\t0.0\t0\t15737\n",
      "19\t17551\t0.0\t0\t17551\n",
      "20\t17836\t0.0\t0\t17836\n",
      "21\t16148\t0.0\t0\t16148\n",
      "22\t16198\t0.0\t0\t16198\n",
      "23\t14185\t0.0\t0\t14185\n",
      "24\t12\t0.0\t0\t12\n",
      "25\t1\t0.0\t0\t1\n",
      "27\t2\t0.0\t0\t2\n",
      "31\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 91926 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t25\t0.0\t0\t25\n",
      "21\t17057\t0.0\t0\t17057\n",
      "22\t16405\t0.0\t0\t16405\n",
      "23\t15378\t0.0\t0\t15378\n",
      "24\t15588\t0.0\t0\t15588\n",
      "25\t14038\t0.0\t0\t14038\n",
      "26\t13408\t0.0\t0\t13408\n",
      "27\t14\t0.0\t0\t14\n",
      "28\t2\t0.0\t0\t2\n",
      "44\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV14_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV14_23_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV14_22_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV14_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV14_22_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV14_23_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV14_22_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV14_23_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.428 s (3.369 µs/read; 17.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            423,732\n",
      "  Read 1 with adapter:                 408,020 (96.3%)\n",
      "  Read 2 with adapter:                 383,994 (90.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           53,643 (12.7%)\n",
      "Pairs written (passing filters):       370,089 (87.3%)\n",
      "\n",
      "Total basepairs processed:   216,103,320 bp\n",
      "  Read 1:   129,238,260 bp\n",
      "  Read 2:    86,865,060 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    172,527,574 bp (79.8%)\n",
      "  Read 1:   105,304,002 bp\n",
      "  Read 2:    67,223,572 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 408020 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t11\t6.5\t0\t11\n",
      "9\t4\t1.6\t0\t4\n",
      "10\t5\t0.4\t0\t5\n",
      "11\t9\t0.1\t0\t9\n",
      "12\t13\t0.0\t0\t13\n",
      "13\t6\t0.0\t0\t6\n",
      "14\t12\t0.0\t0\t12\n",
      "15\t10\t0.0\t0\t10\n",
      "16\t20\t0.0\t0\t20\n",
      "17\t96\t0.0\t0\t96\n",
      "18\t62304\t0.0\t0\t62304\n",
      "19\t73416\t0.0\t0\t73416\n",
      "20\t75244\t0.0\t0\t75244\n",
      "21\t67716\t0.0\t0\t67716\n",
      "22\t70105\t0.0\t0\t70105\n",
      "23\t58993\t0.0\t0\t58993\n",
      "24\t48\t0.0\t0\t48\n",
      "25\t1\t0.0\t0\t1\n",
      "26\t2\t0.0\t0\t2\n",
      "27\t2\t0.0\t0\t2\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 383994 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.4\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t4\t0.0\t0\t4\n",
      "15\t7\t0.0\t0\t7\n",
      "16\t2\t0.0\t0\t2\n",
      "18\t11\t0.0\t0\t11\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t119\t0.0\t0\t119\n",
      "21\t71548\t0.0\t0\t71548\n",
      "22\t68838\t0.0\t0\t68838\n",
      "23\t64512\t0.0\t0\t64512\n",
      "24\t64225\t0.0\t0\t64225\n",
      "25\t58568\t0.0\t0\t58568\n",
      "26\t56085\t0.0\t0\t56085\n",
      "27\t56\t0.0\t0\t56\n",
      "28\t2\t0.0\t0\t2\n",
      "30\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "47\t4\t0.0\t0\t4\n",
      "48\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV15_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV15_25_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV15_24_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV15_25_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV15_24_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV15_25_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV15_24_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV15_25_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.991 s (3.193 µs/read; 18.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            623,590\n",
      "  Read 1 with adapter:                 600,820 (96.3%)\n",
      "  Read 2 with adapter:                 564,438 (90.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           79,320 (12.7%)\n",
      "Pairs written (passing filters):       544,270 (87.3%)\n",
      "\n",
      "Total basepairs processed:   318,030,900 bp\n",
      "  Read 1:   190,194,950 bp\n",
      "  Read 2:   127,835,950 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    253,727,518 bp (79.8%)\n",
      "  Read 1:   154,864,854 bp\n",
      "  Read 2:    98,862,664 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 600820 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t9.5\t0\t2\n",
      "9\t6\t2.4\t0\t6\n",
      "10\t7\t0.6\t0\t7\n",
      "11\t10\t0.1\t0\t10\n",
      "12\t10\t0.0\t0\t10\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t24\t0.0\t0\t24\n",
      "15\t18\t0.0\t0\t18\n",
      "16\t30\t0.0\t0\t30\n",
      "17\t171\t0.0\t0\t171\n",
      "18\t91895\t0.0\t0\t91895\n",
      "19\t107995\t0.0\t0\t107995\n",
      "20\t111207\t0.0\t0\t111207\n",
      "21\t99290\t0.0\t0\t99290\n",
      "22\t102924\t0.0\t0\t102924\n",
      "23\t87147\t0.0\t0\t87147\n",
      "24\t70\t0.0\t0\t70\n",
      "25\t4\t0.0\t0\t4\n",
      "32\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "38\t2\t0.0\t0\t2\n",
      "47\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "65\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 564438 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.6\t0\t1\n",
      "11\t2\t0.1\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t5\t0.0\t0\t5\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t18\t0.0\t0\t18\n",
      "16\t6\t0.0\t0\t6\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t8\t0.0\t0\t8\n",
      "19\t15\t0.0\t0\t15\n",
      "20\t192\t0.0\t0\t192\n",
      "21\t105420\t0.0\t0\t105420\n",
      "22\t100870\t0.0\t0\t100870\n",
      "23\t95117\t0.0\t0\t95117\n",
      "24\t94346\t0.0\t0\t94346\n",
      "25\t85666\t0.0\t0\t85666\n",
      "26\t82681\t0.0\t0\t82681\n",
      "27\t68\t0.0\t0\t68\n",
      "28\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "32\t2\t0.0\t0\t2\n",
      "33\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "40\t2\t0.0\t0\t2\n",
      "41\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV16_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV16_27_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV16_26_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV16_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV16_26_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV16_27_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV16_26_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV16_27_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 5.595 s (5.060 µs/read; 11.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:          1,105,853\n",
      "  Read 1 with adapter:               1,069,463 (96.7%)\n",
      "  Read 2 with adapter:               1,024,372 (92.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:          114,526 (10.4%)\n",
      "Pairs written (passing filters):       991,327 (89.6%)\n",
      "\n",
      "Total basepairs processed:   563,985,030 bp\n",
      "  Read 1:   337,285,165 bp\n",
      "  Read 2:   226,699,865 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    462,118,553 bp (81.9%)\n",
      "  Read 1:   282,064,993 bp\n",
      "  Read 2:   180,053,560 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 1069463 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t6\t16.9\t0\t6\n",
      "9\t14\t4.2\t0\t14\n",
      "10\t13\t1.1\t0\t13\n",
      "11\t12\t0.3\t0\t12\n",
      "12\t10\t0.1\t0\t10\n",
      "13\t9\t0.0\t0\t9\n",
      "14\t48\t0.0\t0\t48\n",
      "15\t49\t0.0\t0\t49\n",
      "16\t54\t0.0\t0\t54\n",
      "17\t309\t0.0\t0\t309\n",
      "18\t162891\t0.0\t0\t162891\n",
      "19\t190774\t0.0\t0\t190774\n",
      "20\t197684\t0.0\t0\t197684\n",
      "21\t178184\t0.0\t0\t178184\n",
      "22\t183477\t0.0\t0\t183477\n",
      "23\t155740\t0.0\t0\t155740\n",
      "24\t139\t0.0\t0\t139\n",
      "25\t1\t0.0\t0\t1\n",
      "26\t2\t0.0\t0\t2\n",
      "27\t1\t0.0\t0\t1\n",
      "29\t2\t0.0\t0\t2\n",
      "30\t2\t0.0\t0\t2\n",
      "31\t3\t0.0\t0\t3\n",
      "32\t2\t0.0\t0\t2\n",
      "33\t1\t0.0\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "37\t4\t0.0\t0\t4\n",
      "38\t8\t0.0\t0\t8\n",
      "39\t6\t0.0\t0\t6\n",
      "40\t5\t0.0\t0\t5\n",
      "41\t5\t0.0\t0\t5\n",
      "42\t3\t0.0\t0\t3\n",
      "43\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1024372 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t16.9\t0\t1\n",
      "9\t2\t4.2\t0\t2\n",
      "10\t2\t1.1\t0\t2\n",
      "11\t2\t0.3\t0\t2\n",
      "12\t3\t0.1\t0\t3\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t40\t0.0\t0\t40\n",
      "16\t12\t0.0\t0\t12\n",
      "17\t10\t0.0\t0\t10\n",
      "18\t11\t0.0\t0\t11\n",
      "19\t27\t0.0\t0\t27\n",
      "20\t357\t0.0\t0\t357\n",
      "21\t189983\t0.0\t0\t189983\n",
      "22\t182150\t0.0\t0\t182150\n",
      "23\t172437\t0.0\t0\t172437\n",
      "24\t172638\t0.0\t0\t172638\n",
      "25\t155788\t0.0\t0\t155788\n",
      "26\t150457\t0.0\t0\t150457\n",
      "27\t123\t0.0\t0\t123\n",
      "28\t2\t0.0\t0\t2\n",
      "32\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "37\t2\t0.0\t0\t2\n",
      "38\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "45\t3\t0.0\t0\t3\n",
      "46\t16\t0.0\t0\t16\n",
      "47\t48\t0.0\t0\t48\n",
      "48\t65\t0.0\t0\t65\n",
      "49\t57\t0.0\t0\t57\n",
      "50\t55\t0.0\t0\t55\n",
      "51\t36\t0.0\t0\t36\n",
      "52\t30\t0.0\t0\t30\n",
      "53\t1\t0.0\t0\t1\n",
      "62\t1\t0.0\t0\t1\n",
      "72\t1\t0.0\t0\t1\n",
      "73\t2\t0.0\t0\t2\n",
      "74\t1\t0.0\t0\t1\n",
      "75\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV18_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV18_29_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV18_28_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV18_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV18_28_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV18_29_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV18_28_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV18_29_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.450 s (4.334 µs/read; 13.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            103,840\n",
      "  Read 1 with adapter:                 100,332 (96.6%)\n",
      "  Read 2 with adapter:                  94,903 (91.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           12,070 (11.6%)\n",
      "Pairs written (passing filters):        91,770 (88.4%)\n",
      "\n",
      "Total basepairs processed:    52,958,400 bp\n",
      "  Read 1:    31,671,200 bp\n",
      "  Read 2:    21,287,200 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     42,781,151 bp (80.8%)\n",
      "  Read 1:    26,111,557 bp\n",
      "  Read 2:    16,669,594 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 100332 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t3\t0.4\t0\t3\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t5\t0.0\t0\t5\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t21\t0.0\t0\t21\n",
      "18\t15435\t0.0\t0\t15435\n",
      "19\t17756\t0.0\t0\t17756\n",
      "20\t18590\t0.0\t0\t18590\n",
      "21\t16775\t0.0\t0\t16775\n",
      "22\t17038\t0.0\t0\t17038\n",
      "23\t14689\t0.0\t0\t14689\n",
      "24\t14\t0.0\t0\t14\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 94903 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t0\t2\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t35\t0.0\t0\t35\n",
      "21\t17696\t0.0\t0\t17696\n",
      "22\t16966\t0.0\t0\t16966\n",
      "23\t16060\t0.0\t0\t16060\n",
      "24\t15970\t0.0\t0\t15970\n",
      "25\t14267\t0.0\t0\t14267\n",
      "26\t13891\t0.0\t0\t13891\n",
      "27\t7\t0.0\t0\t7\n",
      "37\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV21_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV21_31_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV21_30_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV21_31_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV21_30_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV21_31_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV21_30_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV21_31_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.210 s (6.361 µs/read; 9.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,967\n",
      "  Read 1 with adapter:                  31,743 (96.3%)\n",
      "  Read 2 with adapter:                  29,846 (90.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            4,221 (12.8%)\n",
      "Pairs written (passing filters):        28,746 (87.2%)\n",
      "\n",
      "Total basepairs processed:    16,813,170 bp\n",
      "  Read 1:    10,054,935 bp\n",
      "  Read 2:     6,758,235 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     13,401,067 bp (79.7%)\n",
      "  Read 1:     8,179,386 bp\n",
      "  Read 2:     5,221,681 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 31743 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t11\t0.0\t0\t11\n",
      "18\t4831\t0.0\t0\t4831\n",
      "19\t5678\t0.0\t0\t5678\n",
      "20\t6019\t0.0\t0\t6019\n",
      "21\t5227\t0.0\t0\t5227\n",
      "22\t5386\t0.0\t0\t5386\n",
      "23\t4588\t0.0\t0\t4588\n",
      "24\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 29846 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.1\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t14\t0.0\t0\t14\n",
      "21\t5605\t0.0\t0\t5605\n",
      "22\t5319\t0.0\t0\t5319\n",
      "23\t5011\t0.0\t0\t5011\n",
      "24\t5071\t0.0\t0\t5071\n",
      "25\t4520\t0.0\t0\t4520\n",
      "26\t4294\t0.0\t0\t4294\n",
      "27\t4\t0.0\t0\t4\n",
      "30\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV24_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV24_33_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV24_32_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV24_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV24_32_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV24_33_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV24_32_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV24_33_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.291 s (5.219 µs/read; 11.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,667\n",
      "  Read 1 with adapter:                  53,901 (96.8%)\n",
      "  Read 2 with adapter:                  51,290 (92.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,954 (10.7%)\n",
      "Pairs written (passing filters):        49,713 (89.3%)\n",
      "\n",
      "Total basepairs processed:    28,390,170 bp\n",
      "  Read 1:    16,978,435 bp\n",
      "  Read 2:    11,411,735 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     23,175,094 bp (81.6%)\n",
      "  Read 1:    14,144,671 bp\n",
      "  Read 2:     9,030,423 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 53901 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t0.8\t0\t2\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t12\t0.0\t0\t12\n",
      "18\t8227\t0.0\t0\t8227\n",
      "19\t9485\t0.0\t0\t9485\n",
      "20\t9950\t0.0\t0\t9950\n",
      "21\t9047\t0.0\t0\t9047\n",
      "22\t9404\t0.0\t0\t9404\n",
      "23\t7762\t0.0\t0\t7762\n",
      "24\t5\t0.0\t0\t5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 51290 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t0\t1\n",
      "18\t3\t0.0\t0\t3\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t16\t0.0\t0\t16\n",
      "21\t9624\t0.0\t0\t9624\n",
      "22\t9187\t0.0\t0\t9187\n",
      "23\t8606\t0.0\t0\t8606\n",
      "24\t8714\t0.0\t0\t8714\n",
      "25\t7682\t0.0\t0\t7682\n",
      "26\t7445\t0.0\t0\t7445\n",
      "27\t6\t0.0\t0\t6\n",
      "46\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "60\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV25_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV25_35_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV25_34_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV25_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV25_34_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV25_35_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV25_34_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV25_35_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.358 s (5.164 µs/read; 11.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,270\n",
      "  Read 1 with adapter:                  67,056 (96.8%)\n",
      "  Read 2 with adapter:                  63,409 (91.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            7,853 (11.3%)\n",
      "Pairs written (passing filters):        61,417 (88.7%)\n",
      "\n",
      "Total basepairs processed:    35,327,700 bp\n",
      "  Read 1:    21,127,350 bp\n",
      "  Read 2:    14,200,350 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     28,631,860 bp (81.0%)\n",
      "  Read 1:    17,475,307 bp\n",
      "  Read 2:    11,156,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 67056 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.1\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t17\t0.0\t0\t17\n",
      "18\t10263\t0.0\t0\t10263\n",
      "19\t12074\t0.0\t0\t12074\n",
      "20\t12391\t0.0\t0\t12391\n",
      "21\t11134\t0.0\t0\t11134\n",
      "22\t11415\t0.0\t0\t11415\n",
      "23\t9744\t0.0\t0\t9744\n",
      "24\t8\t0.0\t0\t8\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 63409 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t18\t0.0\t0\t18\n",
      "21\t11841\t0.0\t0\t11841\n",
      "22\t11493\t0.0\t0\t11493\n",
      "23\t10787\t0.0\t0\t10787\n",
      "24\t10594\t0.0\t0\t10594\n",
      "25\t9473\t0.0\t0\t9473\n",
      "26\t9183\t0.0\t0\t9183\n",
      "27\t10\t0.0\t0\t10\n",
      "30\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "47\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV26_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV26_37_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV26_36_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV26_37_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV26_36_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV26_37_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV26_36_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV26_37_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.335 s (4.452 µs/read; 13.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,325\n",
      "  Read 1 with adapter:                  72,780 (96.6%)\n",
      "  Read 2 with adapter:                  68,710 (91.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            8,865 (11.8%)\n",
      "Pairs written (passing filters):        66,460 (88.2%)\n",
      "\n",
      "Total basepairs processed:    38,415,750 bp\n",
      "  Read 1:    22,974,125 bp\n",
      "  Read 2:    15,441,625 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     30,982,262 bp (80.6%)\n",
      "  Read 1:    18,909,947 bp\n",
      "  Read 2:    12,072,315 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 72780 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.3\t0\t2\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t22\t0.0\t0\t22\n",
      "18\t11165\t0.0\t0\t11165\n",
      "19\t12948\t0.0\t0\t12948\n",
      "20\t13399\t0.0\t0\t13399\n",
      "21\t11988\t0.0\t0\t11988\n",
      "22\t12718\t0.0\t0\t12718\n",
      "23\t10510\t0.0\t0\t10510\n",
      "24\t11\t0.0\t0\t11\n",
      "34\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 68710 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t4\t0.0\t0\t4\n",
      "18\t1\t0.0\t0\t1\n",
      "20\t21\t0.0\t0\t21\n",
      "21\t12827\t0.0\t0\t12827\n",
      "22\t12290\t0.0\t0\t12290\n",
      "23\t11664\t0.0\t0\t11664\n",
      "24\t11534\t0.0\t0\t11534\n",
      "25\t10412\t0.0\t0\t10412\n",
      "26\t9942\t0.0\t0\t9942\n",
      "27\t14\t0.0\t0\t14\n",
      "47\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV27_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV27_39_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV27_38_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV27_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV27_38_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV27_39_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV27_38_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV27_39_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.287 s (5.272 µs/read; 11.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,395\n",
      "  Read 1 with adapter:                  52,468 (96.5%)\n",
      "  Read 2 with adapter:                  49,396 (90.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            6,691 (12.3%)\n",
      "Pairs written (passing filters):        47,704 (87.7%)\n",
      "\n",
      "Total basepairs processed:    27,741,450 bp\n",
      "  Read 1:    16,590,475 bp\n",
      "  Read 2:    11,150,975 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     22,237,867 bp (80.2%)\n",
      "  Read 1:    13,572,752 bp\n",
      "  Read 2:     8,665,115 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 52468 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t19\t0.0\t0\t19\n",
      "18\t7953\t0.0\t0\t7953\n",
      "19\t9362\t0.0\t0\t9362\n",
      "20\t9616\t0.0\t0\t9616\n",
      "21\t8810\t0.0\t0\t8810\n",
      "22\t8876\t0.0\t0\t8876\n",
      "23\t7824\t0.0\t0\t7824\n",
      "24\t4\t0.0\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 49396 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t4\t0.0\t0\t4\n",
      "20\t16\t0.0\t0\t16\n",
      "21\t9211\t0.0\t0\t9211\n",
      "22\t8861\t0.0\t0\t8861\n",
      "23\t8275\t0.0\t0\t8275\n",
      "24\t8330\t0.0\t0\t8330\n",
      "25\t7442\t0.0\t0\t7442\n",
      "26\t7242\t0.0\t0\t7242\n",
      "27\t4\t0.0\t0\t4\n",
      "46\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV29_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV29_41_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV29_40_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV29_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV29_40_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV29_41_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV29_40_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV29_41_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.242 s (5.154 µs/read; 11.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,933\n",
      "  Read 1 with adapter:                  45,079 (96.0%)\n",
      "  Read 2 with adapter:                  42,523 (90.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            6,037 (12.9%)\n",
      "Pairs written (passing filters):        40,896 (87.1%)\n",
      "\n",
      "Total basepairs processed:    23,935,830 bp\n",
      "  Read 1:    14,314,565 bp\n",
      "  Read 2:     9,621,265 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     19,064,549 bp (79.6%)\n",
      "  Read 1:    11,636,153 bp\n",
      "  Read 2:     7,428,396 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 45079 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t8\t0.0\t0\t8\n",
      "18\t6860\t0.0\t0\t6860\n",
      "19\t7954\t0.0\t0\t7954\n",
      "20\t8409\t0.0\t0\t8409\n",
      "21\t7531\t0.0\t0\t7531\n",
      "22\t7749\t0.0\t0\t7749\n",
      "23\t6562\t0.0\t0\t6562\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 42523 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t2\t0.0\t0\t2\n",
      "17\t1\t0.0\t0\t1\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t18\t0.0\t0\t18\n",
      "21\t8053\t0.0\t0\t8053\n",
      "22\t7561\t0.0\t0\t7561\n",
      "23\t7030\t0.0\t0\t7030\n",
      "24\t7040\t0.0\t0\t7040\n",
      "25\t6463\t0.0\t0\t6463\n",
      "26\t6348\t0.0\t0\t6348\n",
      "27\t5\t0.0\t0\t5\n",
      "38\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV31_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV31_43_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV31_42_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV31_43_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV31_42_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV31_43_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV31_42_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV31_43_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.411 s (4.412 µs/read; 13.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             93,090\n",
      "  Read 1 with adapter:                  89,883 (96.6%)\n",
      "  Read 2 with adapter:                  84,741 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           11,191 (12.0%)\n",
      "Pairs written (passing filters):        81,899 (88.0%)\n",
      "\n",
      "Total basepairs processed:    47,475,900 bp\n",
      "  Read 1:    28,392,450 bp\n",
      "  Read 2:    19,083,450 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     38,180,217 bp (80.4%)\n",
      "  Read 1:    23,303,199 bp\n",
      "  Read 2:    14,877,018 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 89883 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t2\t0.1\t0\t2\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t6\t0.0\t0\t6\n",
      "17\t24\t0.0\t0\t24\n",
      "18\t13825\t0.0\t0\t13825\n",
      "19\t16046\t0.0\t0\t16046\n",
      "20\t16666\t0.0\t0\t16666\n",
      "21\t14867\t0.0\t0\t14867\n",
      "22\t15293\t0.0\t0\t15293\n",
      "23\t13134\t0.0\t0\t13134\n",
      "24\t8\t0.0\t0\t8\n",
      "25\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 84741 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.4\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t21\t0.0\t0\t21\n",
      "21\t16094\t0.0\t0\t16094\n",
      "22\t15051\t0.0\t0\t15051\n",
      "23\t14230\t0.0\t0\t14230\n",
      "24\t14150\t0.0\t0\t14150\n",
      "25\t12783\t0.0\t0\t12783\n",
      "26\t12392\t0.0\t0\t12392\n",
      "27\t8\t0.0\t0\t8\n",
      "35\t1\t0.0\t0\t1\n",
      "46\t2\t0.0\t0\t2\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV33_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV33_45_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV33_44_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV33_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV33_44_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV33_45_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV33_44_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV33_45_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.058 s (3.509 µs/read; 17.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            301,398\n",
      "  Read 1 with adapter:                 290,615 (96.4%)\n",
      "  Read 2 with adapter:                 274,167 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           36,842 (12.2%)\n",
      "Pairs written (passing filters):       264,556 (87.8%)\n",
      "\n",
      "Total basepairs processed:   153,712,980 bp\n",
      "  Read 1:    91,926,390 bp\n",
      "  Read 2:    61,786,590 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    123,329,846 bp (80.2%)\n",
      "  Read 1:    75,275,168 bp\n",
      "  Read 2:    48,054,678 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 290615 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t4.6\t0\t2\n",
      "9\t1\t1.1\t0\t1\n",
      "10\t4\t0.3\t0\t4\n",
      "11\t4\t0.1\t0\t4\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t12\t0.0\t0\t12\n",
      "15\t10\t0.0\t0\t10\n",
      "16\t10\t0.0\t0\t10\n",
      "17\t69\t0.0\t0\t69\n",
      "18\t44404\t0.0\t0\t44404\n",
      "19\t51852\t0.0\t0\t51852\n",
      "20\t53590\t0.0\t0\t53590\n",
      "21\t48629\t0.0\t0\t48629\n",
      "22\t49844\t0.0\t0\t49844\n",
      "23\t42137\t0.0\t0\t42137\n",
      "24\t38\t0.0\t0\t38\n",
      "33\t2\t0.0\t0\t2\n",
      "41\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 274167 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t7\t0.0\t0\t7\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t2\t0.0\t0\t2\n",
      "18\t3\t0.0\t0\t3\n",
      "19\t7\t0.0\t0\t7\n",
      "20\t84\t0.0\t0\t84\n",
      "21\t51079\t0.0\t0\t51079\n",
      "22\t49117\t0.0\t0\t49117\n",
      "23\t46345\t0.0\t0\t46345\n",
      "24\t45884\t0.0\t0\t45884\n",
      "25\t41438\t0.0\t0\t41438\n",
      "26\t40146\t0.0\t0\t40146\n",
      "27\t32\t0.0\t0\t32\n",
      "30\t2\t0.0\t0\t2\n",
      "31\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "48\t5\t0.0\t0\t5\n",
      "49\t4\t0.0\t0\t4\n",
      "50\t2\t0.0\t0\t2\n",
      "52\t2\t0.0\t0\t2\n",
      "59\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV36_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV36_47_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV36_46_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV36_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV36_46_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV36_47_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV36_46_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV36_47_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.638 s (4.103 µs/read; 14.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            155,429\n",
      "  Read 1 with adapter:                 150,072 (96.6%)\n",
      "  Read 2 with adapter:                 141,469 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           18,708 (12.0%)\n",
      "Pairs written (passing filters):       136,721 (88.0%)\n",
      "\n",
      "Total basepairs processed:    79,268,790 bp\n",
      "  Read 1:    47,405,845 bp\n",
      "  Read 2:    31,862,945 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     63,737,457 bp (80.4%)\n",
      "  Read 1:    38,902,183 bp\n",
      "  Read 2:    24,835,274 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 150072 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t3\t2.4\t0\t3\n",
      "9\t3\t0.6\t0\t3\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t3\t0.0\t0\t3\n",
      "12\t2\t0.0\t0\t2\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t10\t0.0\t0\t10\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t32\t0.0\t0\t32\n",
      "18\t22916\t0.0\t0\t22916\n",
      "19\t26948\t0.0\t0\t26948\n",
      "20\t27836\t0.0\t0\t27836\n",
      "21\t24877\t0.0\t0\t24877\n",
      "22\t25802\t0.0\t0\t25802\n",
      "23\t21609\t0.0\t0\t21609\n",
      "24\t17\t0.0\t0\t17\n",
      "30\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 141469 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "19\t5\t0.0\t0\t5\n",
      "20\t51\t0.0\t0\t51\n",
      "21\t26407\t0.0\t0\t26407\n",
      "22\t25369\t0.0\t0\t25369\n",
      "23\t23783\t0.0\t0\t23783\n",
      "24\t23941\t0.0\t0\t23941\n",
      "25\t21560\t0.0\t0\t21560\n",
      "26\t20329\t0.0\t0\t20329\n",
      "27\t18\t0.0\t0\t18\n",
      "30\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV38_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV38_49_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV38_48_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV38_49_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV38_48_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV38_49_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV38_48_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV38_49_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.713 s (3.790 µs/read; 15.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            187,977\n",
      "  Read 1 with adapter:                 181,211 (96.4%)\n",
      "  Read 2 with adapter:                 170,212 (90.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           23,700 (12.6%)\n",
      "Pairs written (passing filters):       164,277 (87.4%)\n",
      "\n",
      "Total basepairs processed:    95,868,270 bp\n",
      "  Read 1:    57,332,985 bp\n",
      "  Read 2:    38,535,285 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     76,580,200 bp (79.9%)\n",
      "  Read 1:    46,741,588 bp\n",
      "  Read 2:    29,838,612 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 181211 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.2\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t5\t0.0\t0\t5\n",
      "14\t8\t0.0\t0\t8\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t47\t0.0\t0\t47\n",
      "18\t27595\t0.0\t0\t27595\n",
      "19\t32278\t0.0\t0\t32278\n",
      "20\t33444\t0.0\t0\t33444\n",
      "21\t30108\t0.0\t0\t30108\n",
      "22\t31228\t0.0\t0\t31228\n",
      "23\t26455\t0.0\t0\t26455\n",
      "24\t25\t0.0\t0\t25\n",
      "29\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 170212 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.7\t0\t1\n",
      "10\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t63\t0.0\t0\t63\n",
      "21\t31256\t0.0\t0\t31256\n",
      "22\t30643\t0.0\t0\t30643\n",
      "23\t28871\t0.0\t0\t28871\n",
      "24\t28521\t0.0\t0\t28521\n",
      "25\t25941\t0.0\t0\t25941\n",
      "26\t24868\t0.0\t0\t24868\n",
      "27\t26\t0.0\t0\t26\n",
      "28\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t3\t0.0\t0\t3\n",
      "51\t2\t0.0\t0\t2\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV41_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV41_51_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV41_50_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV41_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV41_50_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV41_51_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV41_50_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV41_51_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.816 s (3.663 µs/read; 16.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            222,652\n",
      "  Read 1 with adapter:                 215,336 (96.7%)\n",
      "  Read 2 with adapter:                 202,900 (91.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           26,234 (11.8%)\n",
      "Pairs written (passing filters):       196,418 (88.2%)\n",
      "\n",
      "Total basepairs processed:   113,552,520 bp\n",
      "  Read 1:    67,908,860 bp\n",
      "  Read 2:    45,643,660 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     91,567,656 bp (80.6%)\n",
      "  Read 1:    55,889,226 bp\n",
      "  Read 2:    35,678,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 215336 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t3.4\t0\t1\n",
      "9\t1\t0.8\t0\t1\n",
      "10\t3\t0.2\t0\t3\n",
      "11\t4\t0.1\t0\t4\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t9\t0.0\t0\t9\n",
      "15\t11\t0.0\t0\t11\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t48\t0.0\t0\t48\n",
      "18\t33162\t0.0\t0\t33162\n",
      "19\t38706\t0.0\t0\t38706\n",
      "20\t39742\t0.0\t0\t39742\n",
      "21\t35857\t0.0\t0\t35857\n",
      "22\t36670\t0.0\t0\t36670\n",
      "23\t31080\t0.0\t0\t31080\n",
      "24\t29\t0.0\t0\t29\n",
      "25\t2\t0.0\t0\t2\n",
      "29\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 202900 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t4\t0.0\t0\t4\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t59\t0.0\t0\t59\n",
      "21\t37957\t0.0\t0\t37957\n",
      "22\t36161\t0.0\t0\t36161\n",
      "23\t34271\t0.0\t0\t34271\n",
      "24\t34258\t0.0\t0\t34258\n",
      "25\t30720\t0.0\t0\t30720\n",
      "26\t29431\t0.0\t0\t29431\n",
      "27\t23\t0.0\t0\t23\n",
      "30\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV42_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV42_53_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV42_52_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV42_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV42_52_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV42_53_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV42_52_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV42_53_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.010 s (3.734 µs/read; 16.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            270,375\n",
      "  Read 1 with adapter:                 261,295 (96.6%)\n",
      "  Read 2 with adapter:                 246,798 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           31,619 (11.7%)\n",
      "Pairs written (passing filters):       238,756 (88.3%)\n",
      "\n",
      "Total basepairs processed:   137,891,250 bp\n",
      "  Read 1:    82,464,375 bp\n",
      "  Read 2:    55,426,875 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    111,303,897 bp (80.7%)\n",
      "  Read 1:    67,935,411 bp\n",
      "  Read 2:    43,368,486 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 261295 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t4.1\t0\t2\n",
      "9\t4\t1.0\t0\t4\n",
      "10\t4\t0.3\t0\t4\n",
      "11\t7\t0.1\t0\t7\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t10\t0.0\t0\t10\n",
      "15\t5\t0.0\t0\t5\n",
      "16\t8\t0.0\t0\t8\n",
      "17\t60\t0.0\t0\t60\n",
      "18\t40047\t0.0\t0\t40047\n",
      "19\t46651\t0.0\t0\t46651\n",
      "20\t48533\t0.0\t0\t48533\n",
      "21\t43542\t0.0\t0\t43542\n",
      "22\t44632\t0.0\t0\t44632\n",
      "23\t37756\t0.0\t0\t37756\n",
      "24\t30\t0.0\t0\t30\n",
      "29\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 246798 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t4\t0.1\t0\t4\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t2\t0.0\t0\t2\n",
      "15\t14\t0.0\t0\t14\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t3\t0.0\t0\t3\n",
      "19\t6\t0.0\t0\t6\n",
      "20\t79\t0.0\t0\t79\n",
      "21\t45943\t0.0\t0\t45943\n",
      "22\t44323\t0.0\t0\t44323\n",
      "23\t41515\t0.0\t0\t41515\n",
      "24\t41343\t0.0\t0\t41343\n",
      "25\t37461\t0.0\t0\t37461\n",
      "26\t36062\t0.0\t0\t36062\n",
      "27\t32\t0.0\t0\t32\n",
      "28\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "40\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV44_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV44_55_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV44_54_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV44_55_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV44_54_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV44_55_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV44_54_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV44_55_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.925 s (3.990 µs/read; 15.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            231,857\n",
      "  Read 1 with adapter:                 224,074 (96.6%)\n",
      "  Read 2 with adapter:                 210,472 (90.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           28,231 (12.2%)\n",
      "Pairs written (passing filters):       203,626 (87.8%)\n",
      "\n",
      "Total basepairs processed:   118,247,070 bp\n",
      "  Read 1:    70,716,385 bp\n",
      "  Read 2:    47,530,685 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     94,928,202 bp (80.3%)\n",
      "  Read 1:    57,939,398 bp\n",
      "  Read 2:    36,988,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 224074 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t3.5\t0\t1\n",
      "9\t2\t0.9\t0\t2\n",
      "11\t6\t0.1\t0\t6\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t11\t0.0\t0\t11\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t15\t0.0\t0\t15\n",
      "17\t59\t0.0\t0\t59\n",
      "18\t34358\t0.0\t0\t34358\n",
      "19\t39887\t0.0\t0\t39887\n",
      "20\t41809\t0.0\t0\t41809\n",
      "21\t37279\t0.0\t0\t37279\n",
      "22\t38275\t0.0\t0\t38275\n",
      "23\t32325\t0.0\t0\t32325\n",
      "24\t35\t0.0\t0\t35\n",
      "26\t1\t0.0\t0\t1\n",
      "27\t2\t0.0\t0\t2\n",
      "41\t1\t0.0\t0\t1\n",
      "44\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 210472 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.9\t0\t1\n",
      "11\t3\t0.1\t0\t3\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "18\t5\t0.0\t0\t5\n",
      "19\t5\t0.0\t0\t5\n",
      "20\t72\t0.0\t0\t72\n",
      "21\t39469\t0.0\t0\t39469\n",
      "22\t37823\t0.0\t0\t37823\n",
      "23\t35555\t0.0\t0\t35555\n",
      "24\t35152\t0.0\t0\t35152\n",
      "25\t31785\t0.0\t0\t31785\n",
      "26\t30566\t0.0\t0\t30566\n",
      "27\t22\t0.0\t0\t22\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t3\t0.0\t0\t3\n",
      "32\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "47\t2\t0.0\t0\t2\n",
      "59\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV45_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV45_57_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV45_56_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV45_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV45_56_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV45_57_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV45_56_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV45_57_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.741 s (4.290 µs/read; 13.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            172,758\n",
      "  Read 1 with adapter:                 166,743 (96.5%)\n",
      "  Read 2 with adapter:                 156,567 (90.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           21,524 (12.5%)\n",
      "Pairs written (passing filters):       151,234 (87.5%)\n",
      "\n",
      "Total basepairs processed:    88,106,580 bp\n",
      "  Read 1:    52,691,190 bp\n",
      "  Read 2:    35,415,390 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     70,500,598 bp (80.0%)\n",
      "  Read 1:    43,029,758 bp\n",
      "  Read 2:    27,470,840 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 166743 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.6\t0\t1\n",
      "9\t1\t0.7\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t8\t0.0\t0\t8\n",
      "15\t6\t0.0\t0\t6\n",
      "16\t9\t0.0\t0\t9\n",
      "17\t48\t0.0\t0\t48\n",
      "18\t25449\t0.0\t0\t25449\n",
      "19\t29475\t0.0\t0\t29475\n",
      "20\t30641\t0.0\t0\t30641\n",
      "21\t27911\t0.0\t0\t27911\n",
      "22\t28738\t0.0\t0\t28738\n",
      "23\t24428\t0.0\t0\t24428\n",
      "24\t20\t0.0\t0\t20\n",
      "26\t2\t0.0\t0\t2\n",
      "27\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 156567 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.7\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t4\t0.0\t0\t4\n",
      "19\t7\t0.0\t0\t7\n",
      "20\t53\t0.0\t0\t53\n",
      "21\t29296\t0.0\t0\t29296\n",
      "22\t27964\t0.0\t0\t27964\n",
      "23\t26444\t0.0\t0\t26444\n",
      "24\t26185\t0.0\t0\t26185\n",
      "25\t23614\t0.0\t0\t23614\n",
      "26\t22967\t0.0\t0\t22967\n",
      "27\t18\t0.0\t0\t18\n",
      "31\t1\t0.0\t0\t1\n",
      "51\t1\t0.0\t0\t1\n",
      "60\t1\t0.0\t0\t1\n",
      "64\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV46_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV46_59_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV46_58_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV46_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV46_58_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV46_59_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV46_58_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV46_59_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.556 s (3.648 µs/read; 16.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            152,443\n",
      "  Read 1 with adapter:                 146,725 (96.2%)\n",
      "  Read 2 with adapter:                 136,900 (89.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           20,529 (13.5%)\n",
      "Pairs written (passing filters):       131,914 (86.5%)\n",
      "\n",
      "Total basepairs processed:    77,745,930 bp\n",
      "  Read 1:    46,495,115 bp\n",
      "  Read 2:    31,250,815 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     61,495,958 bp (79.1%)\n",
      "  Read 1:    37,534,335 bp\n",
      "  Read 2:    23,961,623 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 146725 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.3\t0\t1\n",
      "9\t2\t0.6\t0\t2\n",
      "10\t1\t0.1\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t44\t0.0\t0\t44\n",
      "18\t22343\t0.0\t0\t22343\n",
      "19\t26129\t0.0\t0\t26129\n",
      "20\t27385\t0.0\t0\t27385\n",
      "21\t24198\t0.0\t0\t24198\n",
      "22\t25272\t0.0\t0\t25272\n",
      "23\t21308\t0.0\t0\t21308\n",
      "24\t25\t0.0\t0\t25\n",
      "25\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 136900 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t6\t0.0\t0\t6\n",
      "19\t9\t0.0\t0\t9\n",
      "20\t44\t0.0\t0\t44\n",
      "21\t25576\t0.0\t0\t25576\n",
      "22\t24507\t0.0\t0\t24507\n",
      "23\t23160\t0.0\t0\t23160\n",
      "24\t22848\t0.0\t0\t22848\n",
      "25\t20770\t0.0\t0\t20770\n",
      "26\t19949\t0.0\t0\t19949\n",
      "27\t14\t0.0\t0\t14\n",
      "37\t1\t0.0\t0\t1\n",
      "48\t3\t0.0\t0\t3\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t2\t0.0\t0\t2\n",
      "74\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV47_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV47_61_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV47_60_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV47_61_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV47_60_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV47_61_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV47_60_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV47_61_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n",
      "Finished in 0.054 s (317.309 µs/read; 0.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                171\n",
      "  Read 1 with adapter:                     168 (98.2%)\n",
      "  Read 2 with adapter:                      86 (50.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:               85 (49.7%)\n",
      "Pairs written (passing filters):            86 (50.3%)\n",
      "\n",
      "Total basepairs processed:        87,210 bp\n",
      "  Read 1:        52,155 bp\n",
      "  Read 2:        35,055 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):         40,125 bp (46.0%)\n",
      "  Read 1:        24,494 bp\n",
      "  Read 2:        15,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 168 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t34\t0.0\t0\t34\n",
      "19\t28\t0.0\t0\t28\n",
      "20\t31\t0.0\t0\t31\n",
      "21\t28\t0.0\t0\t28\n",
      "22\t27\t0.0\t0\t27\n",
      "23\t20\t0.0\t0\t20\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 86 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t14\t0.0\t0\t14\n",
      "22\t17\t0.0\t0\t17\n",
      "23\t16\t0.0\t0\t16\n",
      "24\t23\t0.0\t0\t23\n",
      "25\t5\t0.0\t0\t5\n",
      "26\t11\t0.0\t0\t11\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV48_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV48_63_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV48_62_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV48_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV48_62_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV48_63_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV48_62_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV48_63_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.057 s (113.842 µs/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                498\n",
      "  Read 1 with adapter:                     476 (95.6%)\n",
      "  Read 2 with adapter:                     398 (79.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:              112 (22.5%)\n",
      "Pairs written (passing filters):           386 (77.5%)\n",
      "\n",
      "Total basepairs processed:       253,980 bp\n",
      "  Read 1:       151,890 bp\n",
      "  Read 2:       102,090 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):        179,975 bp (70.9%)\n",
      "  Read 1:       109,858 bp\n",
      "  Read 2:        70,117 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 476 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t83\t0.0\t0\t83\n",
      "19\t102\t0.0\t0\t102\n",
      "20\t76\t0.0\t0\t76\n",
      "21\t77\t0.0\t0\t77\n",
      "22\t73\t0.0\t0\t73\n",
      "23\t64\t0.0\t0\t64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 398 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t0\t1\n",
      "21\t73\t0.0\t0\t73\n",
      "22\t74\t0.0\t0\t74\n",
      "23\t67\t0.0\t0\t67\n",
      "24\t61\t0.0\t0\t61\n",
      "25\t65\t0.0\t0\t65\n",
      "26\t57\t0.0\t0\t57\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV49_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV49_65_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV49_64_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV49_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV49_64_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV49_65_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV49_64_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV49_65_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.731 s (6.378 µs/read; 9.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,604\n",
      "  Read 1 with adapter:                 110,251 (96.2%)\n",
      "  Read 2 with adapter:                 103,663 (90.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           14,761 (12.9%)\n",
      "Pairs written (passing filters):        99,843 (87.1%)\n",
      "\n",
      "Total basepairs processed:    58,448,040 bp\n",
      "  Read 1:    34,954,220 bp\n",
      "  Read 2:    23,493,820 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     46,544,711 bp (79.6%)\n",
      "  Read 1:    28,409,180 bp\n",
      "  Read 2:    18,135,531 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 110251 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.7\t0\t1\n",
      "9\t3\t0.4\t0\t3\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t8\t0.0\t0\t8\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t4\t0.0\t0\t4\n",
      "17\t31\t0.0\t0\t31\n",
      "18\t16706\t0.0\t0\t16706\n",
      "19\t19844\t0.0\t0\t19844\n",
      "20\t20403\t0.0\t0\t20403\n",
      "21\t18571\t0.0\t0\t18571\n",
      "22\t18839\t0.0\t0\t18839\n",
      "23\t15819\t0.0\t0\t15819\n",
      "24\t16\t0.0\t0\t16\n",
      "39\t1\t0.0\t0\t1\n",
      "41\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 103663 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t6\t0.0\t0\t6\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t50\t0.0\t0\t50\n",
      "21\t19381\t0.0\t0\t19381\n",
      "22\t18435\t0.0\t0\t18435\n",
      "23\t17552\t0.0\t0\t17552\n",
      "24\t17394\t0.0\t0\t17394\n",
      "25\t15458\t0.0\t0\t15458\n",
      "26\t15361\t0.0\t0\t15361\n",
      "27\t8\t0.0\t0\t8\n",
      "28\t1\t0.0\t0\t1\n",
      "41\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "47\t2\t0.0\t0\t2\n",
      "49\t1\t0.0\t0\t1\n",
      "50\t3\t0.0\t0\t3\n",
      "51\t1\t0.0\t0\t1\n",
      "57\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV56_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV56_67_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV56_66_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV56_67_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV56_66_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV56_67_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV56_66_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV56_67_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.314 s (3.718 µs/read; 16.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            353,316\n",
      "  Read 1 with adapter:                 342,030 (96.8%)\n",
      "  Read 2 with adapter:                 325,108 (92.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           38,358 (10.9%)\n",
      "Pairs written (passing filters):       314,958 (89.1%)\n",
      "\n",
      "Total basepairs processed:   180,191,160 bp\n",
      "  Read 1:   107,761,380 bp\n",
      "  Read 2:    72,429,780 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    146,828,666 bp (81.5%)\n",
      "  Read 1:    89,618,041 bp\n",
      "  Read 2:    57,210,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 342030 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t5.4\t0\t1\n",
      "9\t3\t1.3\t0\t3\n",
      "10\t5\t0.3\t0\t5\n",
      "11\t4\t0.1\t0\t4\n",
      "12\t6\t0.0\t0\t6\n",
      "13\t6\t0.0\t0\t6\n",
      "14\t12\t0.0\t0\t12\n",
      "15\t11\t0.0\t0\t11\n",
      "16\t10\t0.0\t0\t10\n",
      "17\t83\t0.0\t0\t83\n",
      "18\t52942\t0.0\t0\t52942\n",
      "19\t60975\t0.0\t0\t60975\n",
      "20\t62892\t0.0\t0\t62892\n",
      "21\t56888\t0.0\t0\t56888\n",
      "22\t58134\t0.0\t0\t58134\n",
      "23\t50010\t0.0\t0\t50010\n",
      "24\t44\t0.0\t0\t44\n",
      "27\t1\t0.0\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "43\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 325108 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t2\t0.3\t0\t2\n",
      "11\t2\t0.1\t0\t2\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t4\t0.0\t0\t4\n",
      "15\t9\t0.0\t0\t9\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t6\t0.0\t0\t6\n",
      "19\t8\t0.0\t0\t8\n",
      "20\t113\t0.0\t0\t113\n",
      "21\t60331\t0.0\t0\t60331\n",
      "22\t58492\t0.0\t0\t58492\n",
      "23\t54833\t0.0\t0\t54833\n",
      "24\t54612\t0.0\t0\t54612\n",
      "25\t49272\t0.0\t0\t49272\n",
      "26\t47377\t0.0\t0\t47377\n",
      "27\t33\t0.0\t0\t33\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV58_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV58_69_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV58_68_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV58_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV58_68_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV58_69_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV58_68_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV58_69_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.472 s (3.968 µs/read; 15.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            118,952\n",
      "  Read 1 with adapter:                 114,919 (96.6%)\n",
      "  Read 2 with adapter:                 108,952 (91.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           13,586 (11.4%)\n",
      "Pairs written (passing filters):       105,366 (88.6%)\n",
      "\n",
      "Total basepairs processed:    60,665,520 bp\n",
      "  Read 1:    36,280,360 bp\n",
      "  Read 2:    24,385,160 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     49,121,832 bp (81.0%)\n",
      "  Read 1:    29,981,127 bp\n",
      "  Read 2:    19,140,705 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 114919 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t2\t0.5\t0\t2\n",
      "10\t2\t0.1\t0\t2\n",
      "11\t3\t0.0\t0\t3\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t31\t0.0\t0\t31\n",
      "18\t17839\t0.0\t0\t17839\n",
      "19\t20349\t0.0\t0\t20349\n",
      "20\t21494\t0.0\t0\t21494\n",
      "21\t19099\t0.0\t0\t19099\n",
      "22\t19544\t0.0\t0\t19544\n",
      "23\t16530\t0.0\t0\t16530\n",
      "24\t18\t0.0\t0\t18\n",
      "35\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 108952 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "17\t2\t0.0\t0\t2\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t38\t0.0\t0\t38\n",
      "21\t20525\t0.0\t0\t20525\n",
      "22\t19692\t0.0\t0\t19692\n",
      "23\t18449\t0.0\t0\t18449\n",
      "24\t18157\t0.0\t0\t18157\n",
      "25\t16421\t0.0\t0\t16421\n",
      "26\t15654\t0.0\t0\t15654\n",
      "27\t7\t0.0\t0\t7\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV63_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV63_71_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV63_70_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV63_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV63_70_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV63_71_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV63_70_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV63_71_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.205 s (5.966 µs/read; 10.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,437\n",
      "  Read 1 with adapter:                  33,100 (96.1%)\n",
      "  Read 2 with adapter:                  30,954 (89.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            4,650 (13.5%)\n",
      "Pairs written (passing filters):        29,787 (86.5%)\n",
      "\n",
      "Total basepairs processed:    17,562,870 bp\n",
      "  Read 1:    10,503,285 bp\n",
      "  Read 2:     7,059,585 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     13,885,932 bp (79.1%)\n",
      "  Read 1:     8,475,289 bp\n",
      "  Read 2:     5,410,643 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 33100 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.1\t0\t1\n",
      "10\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t9\t0.0\t0\t9\n",
      "18\t4935\t0.0\t0\t4935\n",
      "19\t5932\t0.0\t0\t5932\n",
      "20\t6094\t0.0\t0\t6094\n",
      "21\t5579\t0.0\t0\t5579\n",
      "22\t5673\t0.0\t0\t5673\n",
      "23\t4858\t0.0\t0\t4858\n",
      "24\t7\t0.0\t0\t7\n",
      "37\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 30954 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t9\t0.0\t0\t9\n",
      "21\t5723\t0.0\t0\t5723\n",
      "22\t5519\t0.0\t0\t5519\n",
      "23\t5355\t0.0\t0\t5355\n",
      "24\t5149\t0.0\t0\t5149\n",
      "25\t4672\t0.0\t0\t4672\n",
      "26\t4519\t0.0\t0\t4519\n",
      "27\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV65_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV65_73_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV65_72_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV65_73_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV65_72_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV65_73_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV65_72_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV65_73_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.488 s (4.255 µs/read; 14.10 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            114,656\n",
      "  Read 1 with adapter:                 110,478 (96.4%)\n",
      "  Read 2 with adapter:                 105,545 (92.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           12,866 (11.2%)\n",
      "Pairs written (passing filters):       101,790 (88.8%)\n",
      "\n",
      "Total basepairs processed:    58,474,560 bp\n",
      "  Read 1:    34,970,080 bp\n",
      "  Read 2:    23,504,480 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     47,452,322 bp (81.2%)\n",
      "  Read 1:    28,962,942 bp\n",
      "  Read 2:    18,489,380 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 110478 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t1.7\t0\t2\n",
      "9\t1\t0.4\t0\t1\n",
      "10\t3\t0.1\t0\t3\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t25\t0.0\t0\t25\n",
      "18\t16864\t0.0\t0\t16864\n",
      "19\t19655\t0.0\t0\t19655\n",
      "20\t20410\t0.0\t0\t20410\n",
      "21\t18596\t0.0\t0\t18596\n",
      "22\t18860\t0.0\t0\t18860\n",
      "23\t16036\t0.0\t0\t16036\n",
      "24\t10\t0.0\t0\t10\n",
      "28\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 105545 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t7\t0.0\t0\t7\n",
      "20\t37\t0.0\t0\t37\n",
      "21\t19737\t0.0\t0\t19737\n",
      "22\t18913\t0.0\t0\t18913\n",
      "23\t17825\t0.0\t0\t17825\n",
      "24\t17489\t0.0\t0\t17489\n",
      "25\t15926\t0.0\t0\t15926\n",
      "26\t15587\t0.0\t0\t15587\n",
      "27\t12\t0.0\t0\t12\n",
      "34\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV66_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV66_75_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV66_74_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV66_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV66_74_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV66_75_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV66_74_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV66_75_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.655 s (3.893 µs/read; 15.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            168,151\n",
      "  Read 1 with adapter:                 162,694 (96.8%)\n",
      "  Read 2 with adapter:                 154,605 (91.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           18,453 (11.0%)\n",
      "Pairs written (passing filters):       149,698 (89.0%)\n",
      "\n",
      "Total basepairs processed:    85,757,010 bp\n",
      "  Read 1:    51,286,055 bp\n",
      "  Read 2:    34,470,955 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     69,787,944 bp (81.4%)\n",
      "  Read 1:    42,594,879 bp\n",
      "  Read 2:    27,193,065 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 162694 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.6\t0\t1\n",
      "9\t3\t0.6\t0\t3\n",
      "10\t3\t0.2\t0\t3\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t4\t0.0\t0\t4\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t7\t0.0\t0\t7\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t11\t0.0\t0\t11\n",
      "17\t57\t0.0\t0\t57\n",
      "18\t24930\t0.0\t0\t24930\n",
      "19\t29017\t0.0\t0\t29017\n",
      "20\t30136\t0.0\t0\t30136\n",
      "21\t27181\t0.0\t0\t27181\n",
      "22\t27787\t0.0\t0\t27787\n",
      "23\t23524\t0.0\t0\t23524\n",
      "24\t22\t0.0\t0\t22\n",
      "27\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "53\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 154605 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t53\t0.0\t0\t53\n",
      "21\t28935\t0.0\t0\t28935\n",
      "22\t27904\t0.0\t0\t27904\n",
      "23\t25955\t0.0\t0\t25955\n",
      "24\t26048\t0.0\t0\t26048\n",
      "25\t23356\t0.0\t0\t23356\n",
      "26\t22319\t0.0\t0\t22319\n",
      "27\t22\t0.0\t0\t22\n",
      "29\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "60\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV67_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV67_77_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV67_76_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV67_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV67_76_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV67_77_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV67_76_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV67_77_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.219 s (4.920 µs/read; 12.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             44,464\n",
      "  Read 1 with adapter:                  42,849 (96.4%)\n",
      "  Read 2 with adapter:                  40,309 (90.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,585 (12.6%)\n",
      "Pairs written (passing filters):        38,879 (87.4%)\n",
      "\n",
      "Total basepairs processed:    22,676,640 bp\n",
      "  Read 1:    13,561,520 bp\n",
      "  Read 2:     9,115,120 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     18,124,512 bp (79.9%)\n",
      "  Read 1:    11,061,903 bp\n",
      "  Read 2:     7,062,609 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 42849 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t5\t0.0\t0\t5\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t5\t0.0\t0\t5\n",
      "17\t12\t0.0\t0\t12\n",
      "18\t6382\t0.0\t0\t6382\n",
      "19\t7586\t0.0\t0\t7586\n",
      "20\t8006\t0.0\t0\t8006\n",
      "21\t7175\t0.0\t0\t7175\n",
      "22\t7362\t0.0\t0\t7362\n",
      "23\t6304\t0.0\t0\t6304\n",
      "24\t6\t0.0\t0\t6\n",
      "46\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 40309 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t14\t0.0\t0\t14\n",
      "21\t7609\t0.0\t0\t7609\n",
      "22\t7196\t0.0\t0\t7196\n",
      "23\t6824\t0.0\t0\t6824\n",
      "24\t6818\t0.0\t0\t6818\n",
      "25\t6041\t0.0\t0\t6041\n",
      "26\t5794\t0.0\t0\t5794\n",
      "27\t8\t0.0\t0\t8\n",
      "47\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV68_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV68_79_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV68_78_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV68_79_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV68_78_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV68_79_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV68_78_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV68_79_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.220 s (5.458 µs/read; 10.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,240\n",
      "  Read 1 with adapter:                  38,682 (96.1%)\n",
      "  Read 2 with adapter:                  36,603 (91.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            5,016 (12.5%)\n",
      "Pairs written (passing filters):        35,224 (87.5%)\n",
      "\n",
      "Total basepairs processed:    20,522,400 bp\n",
      "  Read 1:    12,273,200 bp\n",
      "  Read 2:     8,249,200 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     16,420,794 bp (80.0%)\n",
      "  Read 1:    10,022,252 bp\n",
      "  Read 2:     6,398,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 38682 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.6\t0\t1\n",
      "9\t1\t0.2\t0\t1\n",
      "10\t3\t0.0\t0\t3\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t12\t0.0\t0\t12\n",
      "18\t5793\t0.0\t0\t5793\n",
      "19\t6878\t0.0\t0\t6878\n",
      "20\t7207\t0.0\t0\t7207\n",
      "21\t6537\t0.0\t0\t6537\n",
      "22\t6719\t0.0\t0\t6719\n",
      "23\t5520\t0.0\t0\t5520\n",
      "24\t4\t0.0\t0\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 36603 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "16\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t10\t0.0\t0\t10\n",
      "21\t6932\t0.0\t0\t6932\n",
      "22\t6460\t0.0\t0\t6460\n",
      "23\t6191\t0.0\t0\t6191\n",
      "24\t6200\t0.0\t0\t6200\n",
      "25\t5430\t0.0\t0\t5430\n",
      "26\t5371\t0.0\t0\t5371\n",
      "27\t1\t0.0\t0\t1\n",
      "48\t3\t0.0\t0\t3\n",
      "50\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV69_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV69_81_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV69_80_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV69_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV69_80_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV69_81_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV69_80_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV69_81_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.529 s (4.011 µs/read; 14.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            131,958\n",
      "  Read 1 with adapter:                 127,178 (96.4%)\n",
      "  Read 2 with adapter:                 120,708 (91.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           15,518 (11.8%)\n",
      "Pairs written (passing filters):       116,440 (88.2%)\n",
      "\n",
      "Total basepairs processed:    67,298,580 bp\n",
      "  Read 1:    40,247,190 bp\n",
      "  Read 2:    27,051,390 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     54,283,629 bp (80.7%)\n",
      "  Read 1:    33,131,962 bp\n",
      "  Read 2:    21,151,667 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 127178 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.0\t0\t1\n",
      "10\t4\t0.1\t0\t4\n",
      "11\t4\t0.0\t0\t4\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t4\t0.0\t0\t4\n",
      "15\t10\t0.0\t0\t10\n",
      "16\t4\t0.0\t0\t4\n",
      "17\t35\t0.0\t0\t35\n",
      "18\t19621\t0.0\t0\t19621\n",
      "19\t22635\t0.0\t0\t22635\n",
      "20\t23512\t0.0\t0\t23512\n",
      "21\t21216\t0.0\t0\t21216\n",
      "22\t21761\t0.0\t0\t21761\n",
      "23\t18346\t0.0\t0\t18346\n",
      "24\t21\t0.0\t0\t21\n",
      "25\t2\t0.0\t0\t2\n",
      "29\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 120708 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t3\t0.0\t0\t3\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t2\t0.0\t0\t2\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t4\t0.0\t0\t4\n",
      "20\t45\t0.0\t0\t45\n",
      "21\t22643\t0.0\t0\t22643\n",
      "22\t21675\t0.0\t0\t21675\n",
      "23\t20428\t0.0\t0\t20428\n",
      "24\t20321\t0.0\t0\t20321\n",
      "25\t18210\t0.0\t0\t18210\n",
      "26\t17345\t0.0\t0\t17345\n",
      "27\t14\t0.0\t0\t14\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV71_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV71_83_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV71_82_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV71_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV71_82_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV71_83_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV71_82_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV71_83_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.870 s (4.189 µs/read; 14.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            207,773\n",
      "  Read 1 with adapter:                 200,131 (96.3%)\n",
      "  Read 2 with adapter:                 189,397 (91.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           25,267 (12.2%)\n",
      "Pairs written (passing filters):       182,506 (87.8%)\n",
      "\n",
      "Total basepairs processed:   105,964,230 bp\n",
      "  Read 1:    63,370,765 bp\n",
      "  Read 2:    42,593,465 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     85,084,515 bp (80.3%)\n",
      "  Read 1:    51,931,919 bp\n",
      "  Read 2:    33,152,596 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 200131 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t3\t3.2\t0\t3\n",
      "9\t3\t0.8\t0\t3\n",
      "10\t2\t0.2\t0\t2\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t5\t0.0\t0\t5\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t9\t0.0\t0\t9\n",
      "16\t6\t0.0\t0\t6\n",
      "17\t48\t0.0\t0\t48\n",
      "18\t30826\t0.0\t0\t30826\n",
      "19\t35980\t0.0\t0\t35980\n",
      "20\t37078\t0.0\t0\t37078\n",
      "21\t33404\t0.0\t0\t33404\n",
      "22\t34308\t0.0\t0\t34308\n",
      "23\t28425\t0.0\t0\t28425\n",
      "24\t25\t0.0\t0\t25\n",
      "25\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "55\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 189397 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t2\t0.0\t0\t2\n",
      "15\t6\t0.0\t0\t6\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t75\t0.0\t0\t75\n",
      "21\t35458\t0.0\t0\t35458\n",
      "22\t34087\t0.0\t0\t34087\n",
      "23\t32117\t0.0\t0\t32117\n",
      "24\t31707\t0.0\t0\t31707\n",
      "25\t28338\t0.0\t0\t28338\n",
      "26\t27567\t0.0\t0\t27567\n",
      "27\t25\t0.0\t0\t25\n",
      "28\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "47\t3\t0.0\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV72_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV72_85_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV72_84_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV72_85_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV72_84_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV72_85_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV72_84_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV72_85_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.919 s (3.521 µs/read; 17.04 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            261,138\n",
      "  Read 1 with adapter:                 252,691 (96.8%)\n",
      "  Read 2 with adapter:                 238,491 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           30,124 (11.5%)\n",
      "Pairs written (passing filters):       231,014 (88.5%)\n",
      "\n",
      "Total basepairs processed:   133,180,380 bp\n",
      "  Read 1:    79,647,090 bp\n",
      "  Read 2:    53,533,290 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    107,695,717 bp (80.9%)\n",
      "  Read 1:    65,731,107 bp\n",
      "  Read 2:    41,964,610 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 252691 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t4.0\t0\t1\n",
      "9\t3\t1.0\t0\t3\n",
      "10\t6\t0.2\t0\t6\n",
      "11\t6\t0.1\t0\t6\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t9\t0.0\t0\t9\n",
      "15\t10\t0.0\t0\t10\n",
      "16\t11\t0.0\t0\t11\n",
      "17\t61\t0.0\t0\t61\n",
      "18\t38620\t0.0\t0\t38620\n",
      "19\t45138\t0.0\t0\t45138\n",
      "20\t46710\t0.0\t0\t46710\n",
      "21\t42109\t0.0\t0\t42109\n",
      "22\t43133\t0.0\t0\t43133\n",
      "23\t36841\t0.0\t0\t36841\n",
      "24\t26\t0.0\t0\t26\n",
      "25\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "51\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 238491 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.1\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t4\t0.0\t0\t4\n",
      "19\t5\t0.0\t0\t5\n",
      "20\t74\t0.0\t0\t74\n",
      "21\t44603\t0.0\t0\t44603\n",
      "22\t43099\t0.0\t0\t43099\n",
      "23\t40393\t0.0\t0\t40393\n",
      "24\t39991\t0.0\t0\t39991\n",
      "25\t35877\t0.0\t0\t35877\n",
      "26\t34383\t0.0\t0\t34383\n",
      "27\t31\t0.0\t0\t31\n",
      "30\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "41\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t2\t0.0\t0\t2\n",
      "50\t1\t0.0\t0\t1\n",
      "62\t1\t0.0\t0\t1\n",
      "65\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV74_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV74_87_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV74_86_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV74_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV74_86_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV74_87_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV74_86_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV74_87_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.776 s (3.485 µs/read; 17.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            509,614\n",
      "  Read 1 with adapter:                 492,296 (96.6%)\n",
      "  Read 2 with adapter:                 466,264 (91.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           58,800 (11.5%)\n",
      "Pairs written (passing filters):       450,814 (88.5%)\n",
      "\n",
      "Total basepairs processed:   259,903,140 bp\n",
      "  Read 1:   155,432,270 bp\n",
      "  Read 2:   104,470,870 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    210,162,957 bp (80.9%)\n",
      "  Read 1:   128,275,170 bp\n",
      "  Read 2:    81,887,787 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 492296 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t4\t7.8\t0\t4\n",
      "9\t4\t1.9\t0\t4\n",
      "10\t9\t0.5\t0\t9\n",
      "11\t4\t0.1\t0\t4\n",
      "12\t8\t0.0\t0\t8\n",
      "13\t6\t0.0\t0\t6\n",
      "14\t22\t0.0\t0\t22\n",
      "15\t16\t0.0\t0\t16\n",
      "16\t11\t0.0\t0\t11\n",
      "17\t154\t0.0\t0\t154\n",
      "18\t75744\t0.0\t0\t75744\n",
      "19\t88439\t0.0\t0\t88439\n",
      "20\t90918\t0.0\t0\t90918\n",
      "21\t82173\t0.0\t0\t82173\n",
      "22\t83683\t0.0\t0\t83683\n",
      "23\t71051\t0.0\t0\t71051\n",
      "24\t42\t0.0\t0\t42\n",
      "25\t1\t0.0\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "30\t1\t0.0\t0\t1\n",
      "36\t2\t0.0\t0\t2\n",
      "42\t1\t0.0\t0\t1\n",
      "45\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 466264 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t1.9\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t7\t0.0\t0\t7\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t5\t0.0\t0\t5\n",
      "18\t8\t0.0\t0\t8\n",
      "19\t11\t0.0\t0\t11\n",
      "20\t148\t0.0\t0\t148\n",
      "21\t87207\t0.0\t0\t87207\n",
      "22\t83502\t0.0\t0\t83502\n",
      "23\t78190\t0.0\t0\t78190\n",
      "24\t78154\t0.0\t0\t78154\n",
      "25\t70974\t0.0\t0\t70974\n",
      "26\t67980\t0.0\t0\t67980\n",
      "27\t63\t0.0\t0\t63\n",
      "30\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "38\t2\t0.0\t0\t2\n",
      "40\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "59\t1\t0.0\t0\t1\n",
      "63\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV75_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV75_89_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV75_88_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV75_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV75_88_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV75_89_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV75_88_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV75_89_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.436 s (4.756 µs/read; 12.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            301,831\n",
      "  Read 1 with adapter:                 290,854 (96.4%)\n",
      "  Read 2 with adapter:                 274,130 (90.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           37,433 (12.4%)\n",
      "Pairs written (passing filters):       264,398 (87.6%)\n",
      "\n",
      "Total basepairs processed:   153,933,810 bp\n",
      "  Read 1:    92,058,455 bp\n",
      "  Read 2:    61,875,355 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    123,258,658 bp (80.1%)\n",
      "  Read 1:    75,231,643 bp\n",
      "  Read 2:    48,027,015 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 290854 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t3\t4.6\t0\t3\n",
      "9\t7\t1.2\t0\t7\n",
      "10\t1\t0.3\t0\t1\n",
      "11\t5\t0.1\t0\t5\n",
      "12\t3\t0.0\t0\t3\n",
      "13\t6\t0.0\t0\t6\n",
      "14\t17\t0.0\t0\t17\n",
      "15\t9\t0.0\t0\t9\n",
      "16\t11\t0.0\t0\t11\n",
      "17\t68\t0.0\t0\t68\n",
      "18\t44336\t0.0\t0\t44336\n",
      "19\t52439\t0.0\t0\t52439\n",
      "20\t53569\t0.0\t0\t53569\n",
      "21\t48518\t0.0\t0\t48518\n",
      "22\t49784\t0.0\t0\t49784\n",
      "23\t42049\t0.0\t0\t42049\n",
      "24\t26\t0.0\t0\t26\n",
      "30\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 274130 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t3\t0.3\t0\t3\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t4\t0.0\t0\t4\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t5\t0.0\t0\t5\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t7\t0.0\t0\t7\n",
      "20\t117\t0.0\t0\t117\n",
      "21\t51417\t0.0\t0\t51417\n",
      "22\t49083\t0.0\t0\t49083\n",
      "23\t45978\t0.0\t0\t45978\n",
      "24\t45989\t0.0\t0\t45989\n",
      "25\t41446\t0.0\t0\t41446\n",
      "26\t40037\t0.0\t0\t40037\n",
      "27\t23\t0.0\t0\t23\n",
      "30\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "50\t2\t0.0\t0\t2\n",
      "52\t1\t0.0\t0\t1\n",
      "59\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV76_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV76_91_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV76_90_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV76_91_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV76_90_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV76_91_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV76_90_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV76_91_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 2.886 s (3.631 µs/read; 16.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            794,762\n",
      "  Read 1 with adapter:                 768,264 (96.7%)\n",
      "  Read 2 with adapter:                 729,762 (91.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           88,728 (11.2%)\n",
      "Pairs written (passing filters):       706,034 (88.8%)\n",
      "\n",
      "Total basepairs processed:   405,328,620 bp\n",
      "  Read 1:   242,402,410 bp\n",
      "  Read 2:   162,926,210 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    329,144,050 bp (81.2%)\n",
      "  Read 1:   200,892,226 bp\n",
      "  Read 2:   128,251,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 768264 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t7\t12.1\t0\t7\n",
      "9\t7\t3.0\t0\t7\n",
      "10\t6\t0.8\t0\t6\n",
      "11\t17\t0.2\t0\t17\n",
      "12\t8\t0.0\t0\t8\n",
      "13\t8\t0.0\t0\t8\n",
      "14\t27\t0.0\t0\t27\n",
      "15\t21\t0.0\t0\t21\n",
      "16\t34\t0.0\t0\t34\n",
      "17\t188\t0.0\t0\t188\n",
      "18\t118069\t0.0\t0\t118069\n",
      "19\t136466\t0.0\t0\t136466\n",
      "20\t142148\t0.0\t0\t142148\n",
      "21\t128362\t0.0\t0\t128362\n",
      "22\t131144\t0.0\t0\t131144\n",
      "23\t111666\t0.0\t0\t111666\n",
      "24\t77\t0.0\t0\t77\n",
      "26\t1\t0.0\t0\t1\n",
      "28\t2\t0.0\t0\t2\n",
      "31\t1\t0.0\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "44\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 729762 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t12.1\t0\t1\n",
      "11\t1\t0.2\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t4\t0.0\t0\t4\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t21\t0.0\t0\t21\n",
      "16\t5\t0.0\t0\t5\n",
      "17\t5\t0.0\t0\t5\n",
      "18\t11\t0.0\t0\t11\n",
      "19\t18\t0.0\t0\t18\n",
      "20\t220\t0.0\t0\t220\n",
      "21\t137044\t0.0\t0\t137044\n",
      "22\t131077\t0.0\t0\t131077\n",
      "23\t123132\t0.0\t0\t123132\n",
      "24\t122254\t0.0\t0\t122254\n",
      "25\t109672\t0.0\t0\t109672\n",
      "26\t106193\t0.0\t0\t106193\n",
      "27\t80\t0.0\t0\t80\n",
      "28\t2\t0.0\t0\t2\n",
      "29\t1\t0.0\t0\t1\n",
      "30\t3\t0.0\t0\t3\n",
      "31\t2\t0.0\t0\t2\n",
      "32\t1\t0.0\t0\t1\n",
      "34\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "46\t2\t0.0\t0\t2\n",
      "47\t2\t0.0\t0\t2\n",
      "48\t2\t0.0\t0\t2\n",
      "51\t1\t0.0\t0\t1\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV78_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV78_93_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV78_92_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV78_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV78_92_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV78_93_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV78_92_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV78_93_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n",
      "Finished in 0.057 s (143.818 µs/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                399\n",
      "  Read 1 with adapter:                     384 (96.2%)\n",
      "  Read 2 with adapter:                     190 (47.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:              214 (53.6%)\n",
      "Pairs written (passing filters):           185 (46.4%)\n",
      "\n",
      "Total basepairs processed:       203,490 bp\n",
      "  Read 1:       121,695 bp\n",
      "  Read 2:        81,795 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):         86,295 bp (42.4%)\n",
      "  Read 1:        52,671 bp\n",
      "  Read 2:        33,624 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 384 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t86\t0.0\t0\t86\n",
      "19\t80\t0.0\t0\t80\n",
      "20\t71\t0.0\t0\t71\n",
      "21\t50\t0.0\t0\t50\n",
      "22\t53\t0.0\t0\t53\n",
      "23\t44\t0.0\t0\t44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 190 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t0\t1\n",
      "21\t42\t0.0\t0\t42\n",
      "22\t25\t0.0\t0\t25\n",
      "23\t41\t0.0\t0\t41\n",
      "24\t39\t0.0\t0\t39\n",
      "25\t15\t0.0\t0\t15\n",
      "26\t26\t0.0\t0\t26\n",
      "36\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV80_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV80_95_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV80_94_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV80_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV80_94_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV80_95_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV80_94_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV80_95_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.179 s (6.010 µs/read; 9.98 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,731\n",
      "  Read 1 with adapter:                  28,749 (96.7%)\n",
      "  Read 2 with adapter:                  27,147 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            3,442 (11.6%)\n",
      "Pairs written (passing filters):        26,289 (88.4%)\n",
      "\n",
      "Total basepairs processed:    15,162,810 bp\n",
      "  Read 1:     9,067,955 bp\n",
      "  Read 2:     6,094,855 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     12,256,050 bp (80.8%)\n",
      "  Read 1:     7,480,555 bp\n",
      "  Read 2:     4,775,495 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 28749 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.5\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t11\t0.0\t0\t11\n",
      "18\t4473\t0.0\t0\t4473\n",
      "19\t5157\t0.0\t0\t5157\n",
      "20\t5301\t0.0\t0\t5301\n",
      "21\t4731\t0.0\t0\t4731\n",
      "22\t4955\t0.0\t0\t4955\n",
      "23\t4110\t0.0\t0\t4110\n",
      "24\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 27147 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t5\t0.0\t0\t5\n",
      "21\t5133\t0.0\t0\t5133\n",
      "22\t4830\t0.0\t0\t4830\n",
      "23\t4633\t0.0\t0\t4633\n",
      "24\t4506\t0.0\t0\t4506\n",
      "25\t4092\t0.0\t0\t4092\n",
      "26\t3942\t0.0\t0\t3942\n",
      "27\t3\t0.0\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV81_96_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV81_97_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV81_96_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV81_97_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV81_96_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV81_97_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV81_96_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV81_97_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.609 s (3.867 µs/read; 15.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            157,436\n",
      "  Read 1 with adapter:                 151,775 (96.4%)\n",
      "  Read 2 with adapter:                 142,788 (90.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           19,641 (12.5%)\n",
      "Pairs written (passing filters):       137,795 (87.5%)\n",
      "\n",
      "Total basepairs processed:    80,292,360 bp\n",
      "  Read 1:    48,017,980 bp\n",
      "  Read 2:    32,274,380 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     64,238,969 bp (80.0%)\n",
      "  Read 1:    39,208,173 bp\n",
      "  Read 2:    25,030,796 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 151775 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t2.4\t0\t1\n",
      "9\t4\t0.6\t0\t4\n",
      "10\t3\t0.2\t0\t3\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t5\t0.0\t0\t5\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t7\t0.0\t0\t7\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t8\t0.0\t0\t8\n",
      "17\t34\t0.0\t0\t34\n",
      "18\t23333\t0.0\t0\t23333\n",
      "19\t27408\t0.0\t0\t27408\n",
      "20\t27922\t0.0\t0\t27922\n",
      "21\t24987\t0.0\t0\t24987\n",
      "22\t25983\t0.0\t0\t25983\n",
      "23\t22053\t0.0\t0\t22053\n",
      "24\t14\t0.0\t0\t14\n",
      "26\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 142788 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.6\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t3\t0.0\t0\t3\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t56\t0.0\t0\t56\n",
      "21\t26681\t0.0\t0\t26681\n",
      "22\t25876\t0.0\t0\t25876\n",
      "23\t23855\t0.0\t0\t23855\n",
      "24\t24237\t0.0\t0\t24237\n",
      "25\t21361\t0.0\t0\t21361\n",
      "26\t20678\t0.0\t0\t20678\n",
      "27\t21\t0.0\t0\t21\n",
      "36\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "47\t1\t0.0\t0\t1\n",
      "51\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV82_98_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV82_99_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV82_98_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV82_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV82_98_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV82_99_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV82_98_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV82_99_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 1.400 s (4.097 µs/read; 14.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            341,813\n",
      "  Read 1 with adapter:                 329,803 (96.5%)\n",
      "  Read 2 with adapter:                 312,076 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           40,396 (11.8%)\n",
      "Pairs written (passing filters):       301,417 (88.2%)\n",
      "\n",
      "Total basepairs processed:   174,324,630 bp\n",
      "  Read 1:   104,252,965 bp\n",
      "  Read 2:    70,071,665 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):    140,515,492 bp (80.6%)\n",
      "  Read 1:    85,764,389 bp\n",
      "  Read 2:    54,751,103 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 329803 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t5.2\t0\t2\n",
      "9\t3\t1.3\t0\t3\n",
      "10\t5\t0.3\t0\t5\n",
      "11\t8\t0.1\t0\t8\n",
      "12\t7\t0.0\t0\t7\n",
      "13\t4\t0.0\t0\t4\n",
      "14\t15\t0.0\t0\t15\n",
      "15\t16\t0.0\t0\t16\n",
      "16\t19\t0.0\t0\t19\n",
      "17\t105\t0.0\t0\t105\n",
      "18\t50418\t0.0\t0\t50418\n",
      "19\t58790\t0.0\t0\t58790\n",
      "20\t61232\t0.0\t0\t61232\n",
      "21\t54918\t0.0\t0\t54918\n",
      "22\t56604\t0.0\t0\t56604\n",
      "23\t47622\t0.0\t0\t47622\n",
      "24\t25\t0.0\t0\t25\n",
      "27\t1\t0.0\t0\t1\n",
      "28\t1\t0.0\t0\t1\n",
      "29\t1\t0.0\t0\t1\n",
      "35\t1\t0.0\t0\t1\n",
      "36\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "44\t1\t0.0\t0\t1\n",
      "50\t1\t0.0\t0\t1\n",
      "60\t2\t0.0\t0\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 312076 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.1\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "15\t8\t0.0\t0\t8\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t5\t0.0\t0\t5\n",
      "18\t7\t0.0\t0\t7\n",
      "19\t5\t0.0\t0\t5\n",
      "20\t109\t0.0\t0\t109\n",
      "21\t58253\t0.0\t0\t58253\n",
      "22\t56164\t0.0\t0\t56164\n",
      "23\t52281\t0.0\t0\t52281\n",
      "24\t52246\t0.0\t0\t52246\n",
      "25\t47315\t0.0\t0\t47315\n",
      "26\t45632\t0.0\t0\t45632\n",
      "27\t35\t0.0\t0\t35\n",
      "30\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "33\t1\t0.0\t0\t1\n",
      "37\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "39\t2\t0.0\t0\t2\n",
      "51\t3\t0.0\t0\t3\n",
      "52\t2\t0.0\t0\t2\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV85_100_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV85_101_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV85_100_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV85_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV85_100_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV85_101_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV85_100_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV85_101_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.374 s (4.815 µs/read; 12.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             77,645\n",
      "  Read 1 with adapter:                  75,048 (96.7%)\n",
      "  Read 2 with adapter:                  71,157 (91.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            8,787 (11.3%)\n",
      "Pairs written (passing filters):        68,858 (88.7%)\n",
      "\n",
      "Total basepairs processed:    39,598,950 bp\n",
      "  Read 1:    23,681,725 bp\n",
      "  Read 2:    15,917,225 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     32,102,254 bp (81.1%)\n",
      "  Read 1:    19,594,999 bp\n",
      "  Read 2:    12,507,255 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 75048 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.2\t0\t1\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t3\t0.0\t0\t3\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t26\t0.0\t0\t26\n",
      "18\t11907\t0.0\t0\t11907\n",
      "19\t13491\t0.0\t0\t13491\n",
      "20\t13835\t0.0\t0\t13835\n",
      "21\t12569\t0.0\t0\t12569\n",
      "22\t12501\t0.0\t0\t12501\n",
      "23\t10693\t0.0\t0\t10693\n",
      "24\t10\t0.0\t0\t10\n",
      "27\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 71157 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t2\t0.0\t0\t2\n",
      "12\t1\t0.0\t0\t1\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "19\t4\t0.0\t0\t4\n",
      "20\t30\t0.0\t0\t30\n",
      "21\t13190\t0.0\t0\t13190\n",
      "22\t12667\t0.0\t0\t12667\n",
      "23\t12020\t0.0\t0\t12020\n",
      "24\t11984\t0.0\t0\t11984\n",
      "25\t10844\t0.0\t0\t10844\n",
      "26\t10400\t0.0\t0\t10400\n",
      "27\t11\t0.0\t0\t11\n",
      "52\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV86_102_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV86_103_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV86_102_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV86_103_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV86_102_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV86_103_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV86_102_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV86_103_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.534 s (4.263 µs/read; 14.07 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            125,157\n",
      "  Read 1 with adapter:                 121,600 (97.2%)\n",
      "  Read 2 with adapter:                 114,706 (91.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           13,605 (10.9%)\n",
      "Pairs written (passing filters):       111,552 (89.1%)\n",
      "\n",
      "Total basepairs processed:    63,830,070 bp\n",
      "  Read 1:    38,172,885 bp\n",
      "  Read 2:    25,657,185 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     52,008,195 bp (81.5%)\n",
      "  Read 1:    31,746,025 bp\n",
      "  Read 2:    20,262,170 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 121600 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t3\t1.9\t0\t3\n",
      "11\t1\t0.0\t0\t1\n",
      "14\t5\t0.0\t0\t5\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t4\t0.0\t0\t4\n",
      "17\t37\t0.0\t0\t37\n",
      "18\t19759\t0.0\t0\t19759\n",
      "19\t21957\t0.0\t0\t21957\n",
      "20\t22573\t0.0\t0\t22573\n",
      "21\t19964\t0.0\t0\t19964\n",
      "22\t20197\t0.0\t0\t20197\n",
      "23\t17077\t0.0\t0\t17077\n",
      "24\t18\t0.0\t0\t18\n",
      "29\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 114706 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t2\t0.0\t0\t2\n",
      "18\t3\t0.0\t0\t3\n",
      "19\t5\t0.0\t0\t5\n",
      "20\t36\t0.0\t0\t36\n",
      "21\t21174\t0.0\t0\t21174\n",
      "22\t20717\t0.0\t0\t20717\n",
      "23\t19157\t0.0\t0\t19157\n",
      "24\t19412\t0.0\t0\t19412\n",
      "25\t17430\t0.0\t0\t17430\n",
      "26\t16752\t0.0\t0\t16752\n",
      "27\t12\t0.0\t0\t12\n",
      "49\t1\t0.0\t0\t1\n",
      "59\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV89_104_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV89_105_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV89_104_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV89_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV89_104_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV89_105_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV89_104_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV89_105_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.487 s (4.465 µs/read; 13.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            109,006\n",
      "  Read 1 with adapter:                 105,547 (96.8%)\n",
      "  Read 2 with adapter:                  99,424 (91.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           12,664 (11.6%)\n",
      "Pairs written (passing filters):        96,342 (88.4%)\n",
      "\n",
      "Total basepairs processed:    55,593,060 bp\n",
      "  Read 1:    33,246,830 bp\n",
      "  Read 2:    22,346,230 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     44,915,248 bp (80.8%)\n",
      "  Read 1:    27,416,240 bp\n",
      "  Read 2:    17,499,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 105547 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.4\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "14\t5\t0.0\t0\t5\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t43\t0.0\t0\t43\n",
      "18\t16959\t0.0\t0\t16959\n",
      "19\t18928\t0.0\t0\t18928\n",
      "20\t19428\t0.0\t0\t19428\n",
      "21\t17659\t0.0\t0\t17659\n",
      "22\t17568\t0.0\t0\t17568\n",
      "23\t14935\t0.0\t0\t14935\n",
      "24\t12\t0.0\t0\t12\n",
      "26\t1\t0.0\t0\t1\n",
      "302\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 99424 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t22\t0.0\t0\t22\n",
      "21\t18331\t0.0\t0\t18331\n",
      "22\t17968\t0.0\t0\t17968\n",
      "23\t16454\t0.0\t0\t16454\n",
      "24\t16805\t0.0\t0\t16805\n",
      "25\t15288\t0.0\t0\t15288\n",
      "26\t14536\t0.0\t0\t14536\n",
      "27\t10\t0.0\t0\t10\n",
      "34\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "49\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV90_106_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV90_107_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV90_106_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV90_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV90_106_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV90_107_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV90_106_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV90_107_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.330 s (4.900 µs/read; 12.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,352\n",
      "  Read 1 with adapter:                  65,182 (96.8%)\n",
      "  Read 2 with adapter:                  60,610 (90.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            8,626 (12.8%)\n",
      "Pairs written (passing filters):        58,726 (87.2%)\n",
      "\n",
      "Total basepairs processed:    34,349,520 bp\n",
      "  Read 1:    20,542,360 bp\n",
      "  Read 2:    13,807,160 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     27,379,309 bp (79.7%)\n",
      "  Read 1:    16,712,370 bp\n",
      "  Read 2:    10,666,939 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 65182 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.0\t0\t1\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t3\t0.0\t0\t3\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t21\t0.0\t0\t21\n",
      "18\t10612\t0.0\t0\t10612\n",
      "19\t11742\t0.0\t0\t11742\n",
      "20\t11878\t0.0\t0\t11878\n",
      "21\t10785\t0.0\t0\t10785\n",
      "22\t11014\t0.0\t0\t11014\n",
      "23\t9108\t0.0\t0\t9108\n",
      "24\t6\t0.0\t0\t6\n",
      "25\t2\t0.0\t0\t2\n",
      "33\t1\t0.0\t0\t1\n",
      "38\t1\t0.0\t0\t1\n",
      "39\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 60610 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t20\t0.0\t0\t20\n",
      "21\t11213\t0.0\t0\t11213\n",
      "22\t11050\t0.0\t0\t11050\n",
      "23\t9882\t0.0\t0\t9882\n",
      "24\t10427\t0.0\t0\t10427\n",
      "25\t9161\t0.0\t0\t9161\n",
      "26\t8839\t0.0\t0\t8839\n",
      "27\t11\t0.0\t0\t11\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV92_108_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV92_109_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV92_108_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV92_109_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV92_108_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV92_109_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV92_108_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV92_109_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.264 s (5.167 µs/read; 11.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,059\n",
      "  Read 1 with adapter:                  49,233 (96.4%)\n",
      "  Read 2 with adapter:                  45,856 (89.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            6,782 (13.3%)\n",
      "Pairs written (passing filters):        44,277 (86.7%)\n",
      "\n",
      "Total basepairs processed:    26,040,090 bp\n",
      "  Read 1:    15,572,995 bp\n",
      "  Read 2:    10,467,095 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     20,642,399 bp (79.3%)\n",
      "  Read 1:    12,600,199 bp\n",
      "  Read 2:     8,042,200 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 49233 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.8\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "14\t4\t0.0\t0\t4\n",
      "16\t2\t0.0\t0\t2\n",
      "17\t19\t0.0\t0\t19\n",
      "18\t8004\t0.0\t0\t8004\n",
      "19\t8753\t0.0\t0\t8753\n",
      "20\t8917\t0.0\t0\t8917\n",
      "21\t8207\t0.0\t0\t8207\n",
      "22\t8369\t0.0\t0\t8369\n",
      "23\t6945\t0.0\t0\t6945\n",
      "24\t10\t0.0\t0\t10\n",
      "29\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 45856 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.2\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "15\t4\t0.0\t0\t4\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t18\t0.0\t0\t18\n",
      "21\t8470\t0.0\t0\t8470\n",
      "22\t8179\t0.0\t0\t8179\n",
      "23\t7724\t0.0\t0\t7724\n",
      "24\t7684\t0.0\t0\t7684\n",
      "25\t7111\t0.0\t0\t7111\n",
      "26\t6650\t0.0\t0\t6650\n",
      "27\t6\t0.0\t0\t6\n",
      "44\t1\t0.0\t0\t1\n",
      "48\t3\t0.0\t0\t3\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV93_110_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV93_111_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV93_110_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV93_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV93_110_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV93_111_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV93_110_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV93_111_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.353 s (4.131 µs/read; 14.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,551\n",
      "  Read 1 with adapter:                  82,633 (96.6%)\n",
      "  Read 2 with adapter:                  76,810 (89.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           11,245 (13.1%)\n",
      "Pairs written (passing filters):        74,306 (86.9%)\n",
      "\n",
      "Total basepairs processed:    43,631,010 bp\n",
      "  Read 1:    26,093,055 bp\n",
      "  Read 2:    17,537,955 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     34,642,477 bp (79.4%)\n",
      "  Read 1:    21,145,317 bp\n",
      "  Read 2:    13,497,160 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 82633 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t2\t1.3\t0\t2\n",
      "10\t1\t0.1\t0\t1\n",
      "13\t2\t0.0\t0\t2\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t5\t0.0\t0\t5\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t18\t0.0\t0\t18\n",
      "18\t13359\t0.0\t0\t13359\n",
      "19\t14932\t0.0\t0\t14932\n",
      "20\t14960\t0.0\t0\t14960\n",
      "21\t13502\t0.0\t0\t13502\n",
      "22\t14057\t0.0\t0\t14057\n",
      "23\t11780\t0.0\t0\t11780\n",
      "24\t8\t0.0\t0\t8\n",
      "25\t1\t0.0\t0\t1\n",
      "26\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 76810 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t30\t0.0\t0\t30\n",
      "21\t14270\t0.0\t0\t14270\n",
      "22\t13808\t0.0\t0\t13808\n",
      "23\t12871\t0.0\t0\t12871\n",
      "24\t12933\t0.0\t0\t12933\n",
      "25\t11756\t0.0\t0\t11756\n",
      "26\t11125\t0.0\t0\t11125\n",
      "27\t9\t0.0\t0\t9\n",
      "64\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV94_112_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV94_113_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV94_112_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV94_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV94_112_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV94_113_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV94_112_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV94_113_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.379 s (4.230 µs/read; 14.18 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             89,564\n",
      "  Read 1 with adapter:                  86,639 (96.7%)\n",
      "  Read 2 with adapter:                  80,702 (90.1%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           11,399 (12.7%)\n",
      "Pairs written (passing filters):        78,165 (87.3%)\n",
      "\n",
      "Total basepairs processed:    45,677,640 bp\n",
      "  Read 1:    27,317,020 bp\n",
      "  Read 2:    18,360,620 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     36,442,581 bp (79.8%)\n",
      "  Read 1:    22,243,777 bp\n",
      "  Read 2:    14,198,804 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 86639 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.4\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t4\t0.0\t0\t4\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t3\t0.0\t0\t3\n",
      "17\t24\t0.0\t0\t24\n",
      "18\t13899\t0.0\t0\t13899\n",
      "19\t15612\t0.0\t0\t15612\n",
      "20\t15832\t0.0\t0\t15832\n",
      "21\t14241\t0.0\t0\t14241\n",
      "22\t14830\t0.0\t0\t14830\n",
      "23\t12176\t0.0\t0\t12176\n",
      "24\t9\t0.0\t0\t9\n",
      "25\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 80702 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "10\t1\t0.1\t0\t1\n",
      "12\t2\t0.0\t0\t2\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t1\t0.0\t0\t1\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t18\t0.0\t0\t18\n",
      "21\t15131\t0.0\t0\t15131\n",
      "22\t14526\t0.0\t0\t14526\n",
      "23\t13650\t0.0\t0\t13650\n",
      "24\t13507\t0.0\t0\t13507\n",
      "25\t12097\t0.0\t0\t12097\n",
      "26\t11754\t0.0\t0\t11754\n",
      "27\t7\t0.0\t0\t7\n",
      "40\t1\t0.0\t0\t1\n",
      "46\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV96_114_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV96_115_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV96_114_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV96_115_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV96_114_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV96_115_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV96_114_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV96_115_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.401 s (4.363 µs/read; 13.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             91,864\n",
      "  Read 1 with adapter:                  89,116 (97.0%)\n",
      "  Read 2 with adapter:                  83,757 (91.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:           10,550 (11.5%)\n",
      "Pairs written (passing filters):        81,314 (88.5%)\n",
      "\n",
      "Total basepairs processed:    46,850,640 bp\n",
      "  Read 1:    28,018,520 bp\n",
      "  Read 2:    18,832,120 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     37,910,012 bp (80.9%)\n",
      "  Read 1:    23,139,809 bp\n",
      "  Read 2:    14,770,203 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 89116 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t3\t1.4\t0\t3\n",
      "10\t2\t0.1\t0\t2\n",
      "11\t3\t0.0\t0\t3\n",
      "12\t1\t0.0\t0\t1\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t6\t0.0\t0\t6\n",
      "15\t4\t0.0\t0\t4\n",
      "16\t7\t0.0\t0\t7\n",
      "17\t24\t0.0\t0\t24\n",
      "18\t14321\t0.0\t0\t14321\n",
      "19\t15980\t0.0\t0\t15980\n",
      "20\t16400\t0.0\t0\t16400\n",
      "21\t14881\t0.0\t0\t14881\n",
      "22\t14818\t0.0\t0\t14818\n",
      "23\t12655\t0.0\t0\t12655\n",
      "24\t7\t0.0\t0\t7\n",
      "28\t1\t0.0\t0\t1\n",
      "44\t1\t0.0\t0\t1\n",
      "48\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 83757 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t0\t1\n",
      "15\t2\t0.0\t0\t2\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t2\t0.0\t0\t2\n",
      "19\t3\t0.0\t0\t3\n",
      "20\t37\t0.0\t0\t37\n",
      "21\t15538\t0.0\t0\t15538\n",
      "22\t14912\t0.0\t0\t14912\n",
      "23\t14204\t0.0\t0\t14204\n",
      "24\t14345\t0.0\t0\t14345\n",
      "25\t12540\t0.0\t0\t12540\n",
      "26\t12160\t0.0\t0\t12160\n",
      "27\t11\t0.0\t0\t11\n",
      "41\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV98_116_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV98_117_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV98_116_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV98_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV98_116_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV98_117_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV98_116_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV98_117_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.453 s (5.330 µs/read; 11.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,898\n",
      "  Read 1 with adapter:                  82,359 (97.0%)\n",
      "  Read 2 with adapter:                  77,478 (91.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            9,654 (11.4%)\n",
      "Pairs written (passing filters):        75,244 (88.6%)\n",
      "\n",
      "Total basepairs processed:    43,297,980 bp\n",
      "  Read 1:    25,893,890 bp\n",
      "  Read 2:    17,404,090 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     35,081,886 bp (81.0%)\n",
      "  Read 1:    21,413,944 bp\n",
      "  Read 2:    13,667,942 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 82359 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t1.3\t0\t1\n",
      "9\t1\t0.3\t0\t1\n",
      "10\t1\t0.1\t0\t1\n",
      "11\t1\t0.0\t0\t1\n",
      "12\t1\t0.0\t0\t1\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t1\t0.0\t0\t1\n",
      "17\t23\t0.0\t0\t23\n",
      "18\t13453\t0.0\t0\t13453\n",
      "19\t14926\t0.0\t0\t14926\n",
      "20\t15173\t0.0\t0\t15173\n",
      "21\t13662\t0.0\t0\t13662\n",
      "22\t13631\t0.0\t0\t13631\n",
      "23\t11469\t0.0\t0\t11469\n",
      "24\t12\t0.0\t0\t12\n",
      "26\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 77478 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t3\t0.0\t0\t3\n",
      "18\t1\t0.0\t0\t1\n",
      "19\t2\t0.0\t0\t2\n",
      "20\t31\t0.0\t0\t31\n",
      "21\t14523\t0.0\t0\t14523\n",
      "22\t14003\t0.0\t0\t14003\n",
      "23\t12799\t0.0\t0\t12799\n",
      "24\t13052\t0.0\t0\t13052\n",
      "25\t11823\t0.0\t0\t11823\n",
      "26\t11230\t0.0\t0\t11230\n",
      "27\t9\t0.0\t0\t9\n",
      "31\t1\t0.0\t0\t1\n",
      "32\t1\t0.0\t0\t1\n",
      "\n",
      "Command: cutadapt --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV99_118_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV99_119_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV99_118_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV99_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.4 with Python 3.8.16\n",
      "Command line parameters: --cores 6 --error-rate 0.01 --times 1 --overlap 8 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV99_118_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-1ge5vizk/MV99_119_L001_R2_001.fastq.gz --front CCTACGGGRSGCAGCAG -G GGACTACHVGGGTWTCTAAT --no-indels --match-read-wildcards --discard-untrimmed /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV99_118_L001_R1_001.fastq.gz /tmp/qiime2/lauro/data/3ad54b12-bedb-43b6-acf2-8f4ebb09817e/data/MV99_119_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 6 cores ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.317 s (5.781 µs/read; 10.38 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,891\n",
      "  Read 1 with adapter:                  53,117 (96.8%)\n",
      "  Read 2 with adapter:                  49,172 (89.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs discarded as untrimmed:            7,249 (13.2%)\n",
      "Pairs written (passing filters):        47,642 (86.8%)\n",
      "\n",
      "Total basepairs processed:    27,994,410 bp\n",
      "  Read 1:    16,741,755 bp\n",
      "  Read 2:    11,252,655 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     22,211,054 bp (79.3%)\n",
      "  Read 1:    13,557,440 bp\n",
      "  Read 2:     8,653,614 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: CCTACGGGRSGCAGCAG; Type: regular 5'; Length: 17; Trimmed: 53117 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-17 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t1\t0.8\t0\t1\n",
      "11\t2\t0.0\t0\t2\n",
      "13\t1\t0.0\t0\t1\n",
      "14\t2\t0.0\t0\t2\n",
      "15\t3\t0.0\t0\t3\n",
      "16\t4\t0.0\t0\t4\n",
      "17\t22\t0.0\t0\t22\n",
      "18\t8336\t0.0\t0\t8336\n",
      "19\t9714\t0.0\t0\t9714\n",
      "20\t9881\t0.0\t0\t9881\n",
      "21\t8689\t0.0\t0\t8689\n",
      "22\t8883\t0.0\t0\t8883\n",
      "23\t7570\t0.0\t0\t7570\n",
      "24\t5\t0.0\t0\t5\n",
      "25\t1\t0.0\t0\t1\n",
      "27\t1\t0.0\t0\t1\n",
      "28\t1\t0.0\t0\t1\n",
      "31\t1\t0.0\t0\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 49172 times\n",
      "\n",
      "Minimum overlap: 8\n",
      "No. of allowed errors:\n",
      "1-20 bp: 0\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t0\t1\n",
      "16\t1\t0.0\t0\t1\n",
      "18\t2\t0.0\t0\t2\n",
      "19\t1\t0.0\t0\t1\n",
      "20\t22\t0.0\t0\t22\n",
      "21\t8998\t0.0\t0\t8998\n",
      "22\t8928\t0.0\t0\t8928\n",
      "23\t8292\t0.0\t0\t8292\n",
      "24\t8345\t0.0\t0\t8345\n",
      "25\t7361\t0.0\t0\t7361\n",
      "26\t7219\t0.0\t0\t7219\n",
      "27\t2\t0.0\t0\t2\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/lauro/anaconda3/envs/qiime2-2023.5/lib/python3.8/site-packages/q2_demux/_summarize/_visualizer.py:191: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  context['result_data'] = context['result_data'].append(df)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/lauro/anaconda3/envs/qiime2-2023.5/lib/python3.8/site-packages/q2_demux/_summarize/_visualizer.py:191: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.\n",
      "  context['result_data'] = context['result_data'].append(df)\n"
     ]
    }
   ],
   "source": [
    "demux_file_trim = os.path.join(out_dir, f'demux-{direction}-trim.qza')\n",
    "demux_view_trim = os.path.join(out_dir, f'demux-{direction}-trim.qzv')\n",
    "\n",
    "if trim and (not os.path.isfile(demux_file_trim) or replace_files):\n",
    "    forward_primer = [trim['forward_primer']] # ['CCTACGGGRSGCAGCAG']\n",
    "    reverse_primer = [trim['reverse_primer']] # ['GGACTACHVGGGTWTCTAAT']\n",
    "    forward_reverse_complement = [str(Seq(forward_primer[0]).reverse_complement())]\n",
    "    reverse_reverse_complement = [str(Seq(reverse_primer[0]).reverse_complement())]\n",
    "\n",
    "    res = None\n",
    "    \n",
    "    if n_directions == 1:\n",
    "        res = trim_single(\n",
    "            demultiplexed_sequences=artifact,\n",
    "            front=forward_primer,\n",
    "            adapter=reverse_reverse_complement,\n",
    "            cores=threads,\n",
    "            indels=False,\n",
    "            match_read_wildcards=True,\n",
    "            match_adapter_wildcards=True,\n",
    "            error_rate=0.01,\n",
    "            discard_untrimmed=True,\n",
    "        ).trimmed_sequences\n",
    "    elif n_directions == 2:\n",
    "        res = trim_paired(\n",
    "            demultiplexed_sequences=artifact,\n",
    "            front_f=forward_primer,\n",
    "            front_r=reverse_primer,\n",
    "            # adapter_f=reverse_reverse_complement,\n",
    "            # adapter_r=forward_reverse_complement,\n",
    "            cores=threads,\n",
    "            overlap=trim['overlap'],\n",
    "            indels=False,\n",
    "            match_read_wildcards=True,\n",
    "            match_adapter_wildcards=True,\n",
    "            error_rate=0.01,\n",
    "            discard_untrimmed=True,\n",
    "        ).trimmed_sequences\n",
    "    else:\n",
    "        print(f'ERROR: invalid number of directions {n_directions}')\n",
    "\n",
    "    if res is not None:\n",
    "        res.save(demux_file_trim)\n",
    "        Visualization.save(demux.visualizers.summarize(res).visualization, filepath=demux_view_trim)"
   ]
  }
 ],
 "metadata": {
  "celltoolbar": "Edit Metadata",
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.16"
  },
  "papermill": {
   "default_parameters": {},
   "duration": 2604.502433,
   "end_time": "2024-02-20T20:03:38.844930",
   "environment_variables": {},
   "exception": null,
   "input_path": "nb-templates/step-prepare-data.ipynb",
   "output_path": "/mnt/nupeb/rede-micro/redemicro-jennefer/experiments/jennefer-vs-paired-end-trim/nb-executed-steps/step-prepare-data-jennefer-vs-paired-end-trim.ipynb",
   "parameters": {
    "base_dir": "/mnt/nupeb/rede-micro/redemicro-jennefer",
    "class_col": "class",
    "classifier_file": "/mnt/nupeb/rede-micro/datasets/16S_classifiers_qiime2/silva-138-99-nb-classifier.qza",
    "experiment_name": "jennefer-vs-paired-end-trim",
    "manifest_file": "/mnt/nupeb/rede-micro/redemicro-jennefer/data/manifest.csv",
    "metadata_file": "/mnt/nupeb/rede-micro/redemicro-jennefer/data/metadata.tsv",
    "overlap": 12,
    "phred": 20,
    "replace_files": false,
    "threads": 6,
    "trim": {
     "forward_primer": "CCTACGGGRSGCAGCAG",
     "overlap": 8,
     "reverse_primer": "GGACTACHVGGGTWTCTAAT"
    },
    "trunc_f": 0,
    "trunc_r": 0
   },
   "start_time": "2024-02-20T19:20:14.342497",
   "version": "2.4.0"
  },
  "toc-autonumbering": false,
  "toc-showcode": false,
  "toc-showmarkdowntxt": false
 },
 "nbformat": 4,
 "nbformat_minor": 5
}